
# Extracting Pubmed abstracts by scripting NCBI E-utilities in Python

Here, I will show you how NCBI E-Utilities can be used to search for and download Pubmed abstracts. I use Python in conjunction with the NCBI E-Utilities to automate the process, in order to download all the abstracts corresponding to a given search term and simultaneously parse the information contained in each abstract into a data science-friendly format.

## Introduction

When you search Pubmed for a specific topic, you’re presented with thousands of abstracts that contain the keyword you used for your query. Finding the information you’re looking for can get a bit tedious when you have to manually sift through each and every search result (click on the abstract, read it, then click to go back, repeat 100x). Fortunately for you there’s an easier way around this that lets you download hundreds of thousands of full-text abstracts (500 abstracts per page), through which you can easily ctrl-F or scroll through.

Using the NCBI E-Utilities (Entrez Programming Utilities, https://www.ncbi.nlm.nih.gov/books/NBK25499/), you can retrieve and download all the abstracts (along with author info, journal info, and date of publication) associated with a pubmed search. Even better, this tool doesn’t require any software–its completely URL based. You craft a custom URL link and open it in your browser window or in Python to access the abstracts.

There are two ways to use this tool:

1. Manual way: to quickly grab all the abstracts relevant to a certain topic and read through them quickly without having to waste time sifting through the pubmed web interface. This requires no coding experience at all, only a brief tutorial on how to manipulate the URL.
2. Automated way: Program a script in Python to construct the URLs for you, extract all the abstracts, and process them into a data file for downstream analysis. Information from each abstract can be analyzed using natural language processing or RegExes to quickly extract numerical data or qualitative results.

## How the tools work:

The two main E-util functions you will use are **esearch** and **efetch**. Esearch runs a search command on the database and retrieves ID’s for each of the abstracts that a normal search would have given you. The actual information associated with the abstracts does not show up, only the IDs. After running the esearch function, you’re given a **query key** and **web environment ID**, which saves this search in the database in order to be able to “fetch” all the abstracts associated with it. 

In order to retrieve the information associated with the abstract IDs, you have to run the efetch command. The efetch command requires you to input the query key and web environment ID of the search you want to download the abstracts from.

Let’s say I want to search pubmed for "P2RY8", which is a GPCR on B cells.

### 1. Craft your esearch URL

Step 1: Craft the URL that will execute a proper esearch result: http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&term=P2RY8&retmax=50&usehistory=y

* http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi? is the backbone of the esearch function.
* db=pubmed specifies that we will be searching the pubmed database. 
* term=P2RY8 specifies what we will be searching pubmed for. Change this field to whatever you want to search for.
* retmax=50 specifies how many abstracts I want to return using the search.
* usehistory=y will provide you with a QueryKey and WebEnv id that will let you fetch abstracts from this search.
* The “&” signs are just used to separate the different conditions. Make sure to include it starting from after the “db=pubmed” criterion.

Copying and pasting the full URL into my web browser results in this output:

![esearch_web_result.png](attachment:esearch_web_result.png)

### 2. Craft the efetch URL

The next step is to execute an efetch command by constructing a new URL. Using the webenv and querykey information given in the above esearch result, I will type the following efetch command into my browser:

https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pubmed&query_key=1&WebEnv=NCID_1_5757184_130.14.18.48_9001_1579844644_2015135327_0MetA0_S_MegaStore&retstart=0&retmax=50&retmode=text&rettype=abstract

Note: If you’re trying this right now, your esearch will have given you a different webenv variable. Make sure to input YOUR webenv variable in the efetch URL for it to work!

Here is an explanation for each aspect of the link I constructed above:
* http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi? is the backbone for a efetch command. Notice that the only difference from this and an esearch command is the part after “/eutils/”.
* db=pubmed specifies the database, again.
* query_key=1 specifies the number that was given in the “querykey” field in the esearch result.
* webenv=NCID_1_5757184_130.14.18.48_9001_1579844644_2015135327_0MetA0_S_MegaStore specifies the ID that was given in the esearch result. 
* retmode = text specifies that I want the abstracts to be written out in print. 
* rettype=abstract specifies that I want abstracts shown, as opposed to other types of info that can be given from a pubmed search.

After inputting this link, you should observe the following page:

![efetch_web_result.png](attachment:efetch_web_result.png)

Now, I can simply ctrl-F to sift through over three hundred abstracts! You can apply this simple two-step process whenever you’re tasked with searching through absurd amounts of Pubmed results.

Below, I show you how to perform this process in Python.


### References
Sayers E. The E-utilities In-Depth: Parameters, Syntax and More. 2009 May 29. In: Entrez Programming Utilities Help. Bethesda (MD): National Center for Biotechnology Information (US); 2010-.Available from: http://www.ncbi.nlm.nih.gov/books/NBK25499/


```python
import csv
import re
import urllib
import os
from time import sleep
```


```python
query = 'P2RY8'
```


```python
# common settings between esearch and efetch
base_url = 'http://eutils.ncbi.nlm.nih.gov/entrez/eutils/'
db = 'db=pubmed'

# esearch specific settings
search_eutil = 'esearch.fcgi?'
search_term = '&term=' + query
search_usehistory = '&usehistory=y'
search_rettype = '&rettype=json'

search_url = base_url+search_eutil+db+search_term+search_usehistory+search_rettype

```


```python
print(search_url)
```

    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi?db=pubmed&term=P2RY8&usehistory=y&rettype=json



```python
f = urllib.request.urlopen (search_url)
search_data = f.read().decode('utf-8')
```


```python
search_data
```




    '<?xml version="1.0" encoding="UTF-8" ?>\n<!DOCTYPE eSearchResult PUBLIC "-//NLM//DTD esearch 20060628//EN" "https://eutils.ncbi.nlm.nih.gov/eutils/dtd/20060628/esearch.dtd">\n<eSearchResult><Count>56</Count><RetMax>20</RetMax><RetStart>0</RetStart><QueryKey>1</QueryKey><WebEnv>NCID_1_5579798_130.14.18.48_9001_1579842017_1501765481_0MetA0_S_MegaStore</WebEnv><IdList>\n<Id>31496804</Id>\n<Id>30977196</Id>\n<Id>30859636</Id>\n<Id>30842656</Id>\n<Id>30487598</Id>\n<Id>29982439</Id>\n<Id>29786757</Id>\n<Id>29507076</Id>\n<Id>29407587</Id>\n<Id>29228539</Id>\n<Id>29194562</Id>\n<Id>29140408</Id>\n<Id>28866095</Id>\n<Id>28597942</Id>\n<Id>28408464</Id>\n<Id>28395118</Id>\n<Id>28371317</Id>\n<Id>28033648</Id>\n<Id>27959929</Id>\n<Id>27899802</Id>\n</IdList><TranslationSet/><TranslationStack>   <TermSet>    <Term>P2RY8[All Fields]</Term>    <Field>All Fields</Field>    <Count>56</Count>    <Explode>N</Explode>   </TermSet>   <OP>GROUP</OP>  </TranslationStack><QueryTranslation>P2RY8[All Fields]</QueryTranslation></eSearchResult>\n'




```python
total_abstract_count = int(re.findall("<Count>(\d+?)</Count>",search_data)[0])

# obtain webenv and querykey settings for efetch command
fetch_webenv = "&WebEnv=" + re.findall ("<WebEnv>(\S+)<\/WebEnv>", search_data)[0]
fetch_querykey = "&query_key=" + re.findall("<QueryKey>(\d+?)</QueryKey>",search_data)[0]
```


```python
total_abstract_count
```




    56




```python
fetch_webenv
```




    '&WebEnv=NCID_1_5579798_130.14.18.48_9001_1579842017_1501765481_0MetA0_S_MegaStore'




```python
fetch_querykey
```




    '&query_key=1'




```python
# other efetch settings
fetch_eutil = 'efetch.fcgi?'
retmax = 50
retstart = 0
fetch_retstart = "&retstart=" + str(retstart)
fetch_retmax = "&retmax=" + str(retmax)
fetch_retmode = "&retmode=text"
fetch_rettype = "&rettype=abstract"

fetch_url = base_url+fetch_eutil+db+fetch_querykey+fetch_webenv+fetch_retstart+fetch_retmax+fetch_retmode+fetch_rettype
print(fetch_url)
```

    http://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pubmed&query_key=1&WebEnv=NCID_1_5579798_130.14.18.48_9001_1579842017_1501765481_0MetA0_S_MegaStore&retstart=0&retmax=50&retmode=text&rettype=abstract



```python
f = urllib.request.urlopen (fetch_url)
fetch_data = f.read().decode('utf-8')
```




    '1. Cancer Manag Res. 2019 Aug 6;11:7455-7472. doi: 10.2147/CMAR.S201177. eCollection\n2019.\n\nThe rel'




```python
chunks = fetch_data.split("\n\n\n")
```


```python
len(chunks)
```




    50




```python
chunks
```




    ["\n1. Cancer Manag Res. 2019 Aug 6;11:7455-7472. doi: 10.2147/CMAR.S201177. eCollection\n2019.\n\nThe relevance between the immune response-related gene module and clinical traits\nin head and neck squamous cell carcinoma.\n\nSong Y(#)(1), Pan Y(#)(1), Liu J(1).\n\nAuthor information: \n(1)State Key Laboratory of Oral Diseases, National Clinical Research Center for\nOral Diseases, Department of Orthodontics, West China Hospital of Stomatology,\nSichuan University, Chengdu, 610041, People's Republic of China.\n(#)Contributed equally\n\nPurpose: Head and neck squamous cell carcinoma (HNSCC) is the sixth most\nprevalent cancer in the world, accounting for more than 90% of head and neck\nmalignant tumors. However, its molecular mechanism is largely unknown. To help\nelucidate the potential mechanism of HNSCC tumorigenesis, we investigated the\ngene interaction patterns associated with tumorigenesis.\nMethods: Weighted gene co-expression network analysis (WGCNA) can help us to\npredict the intrinsic relationship or correlation between gene expression.\nAdditionally, we further explored the combination of clinical information and\nmodule construction.\nResults: Sixteen modules were constructed, among which the key module most\nclosely associated with clinical information was identified. By analyzing the\ngenes in this module, we found that the latter may be related to the immune\nresponse, inflammatory response and formation of the tumor microenvironment.\nSixteen hub genes were identified-ARHGAP9, SASH3, CORO1A, ITGAL, PPP1R16B,\nTBC1D10C, IL10RA, ITK, AKNA, PRKCB, TRAF3IP3, GIMAP4, CCR7, P2RY8, GIMAP7, and\nSP140. We further validated these genes at the transcriptional and translation\nlevels.\nConclusion: The innovative use of a weighted network to analyze HNSCC samples\nprovides new insights into the molecular mechanism and prognosis of HNSCC.\nAdditionally, the hub genes we identified can be used as biomarkers and\ntherapeutic targets of HNSCC, laying the foundation for the accurate diagnosis\nand treatment of HNSCC in clinical and research in the future.\n\nDOI: 10.2147/CMAR.S201177 \nPMCID: PMC6689548\nPMID: 31496804 \n\nConflict of interest statement: The authors report no conflicts of interest in\nthis work.",
     '2. Immunol Rev. 2019 May;289(1):158-172. doi: 10.1111/imr.12743.\n\nG-protein coupled receptors and ligands that organize humoral immune responses.\n\nLu E(1), Cyster JG(1).\n\nAuthor information: \n(1)Howard Hughes Medical Institute and Department of Microbiology and Immunology,\nUniversity of California San Francisco, San Francisco, California.\n\nB-cell responses are dynamic processes that depend on multiple types of\ninteractions. Rare antigen-specific B cells must encounter antigen and\nspecialized systems are needed-unique to each lymphoid tissue type-to ensure this\nhappens efficiently. Lymphoid tissue barrier cells act to ensure that pathogens, \nwhile being permitted entry for B-cell recognition, are blocked from replication \nor dissemination. T follicular helper (Tfh) cells often need to be primed by\ndendritic cells before supporting B-cell responses. For most responses,\nantigen-specific helper T cells and B cells need to interact, first to initiate\nclonal expansion and the plasmablast response, and later to support the germinal \ncenter (GC) response. Newly formed plasma cells need to travel to supportive\nniches. GC B cells must become confined to the follicle center, organize into\ndark and light zones, and interact with Tfh cells. Memory B cells need to be\npositioned for rapid responses following reinfection. Each of these events\nrequires the actions of multiple G-protein coupled receptors (GPCRs) and their\nligands, including chemokines and lipid mediators. This review will focus on the \nguidance cue code underlying B-cell immunity, with an emphasis on findings from\nour laboratory and on newer advances in related areas. We will discuss our recent\nidentification of geranylgeranyl-glutathione as a ligand for P2RY8. Our goal is\nto provide the reader with a focused knowledge about the GPCRs guiding B-cell\nresponses and how they might be therapeutic targets, while also providing\nexamples of how multiple types of GPCRs can cooperate or act iteratively to\ncontrol cell behavior.\n\n© 2019 John Wiley & Sons A/S. Published by John Wiley & Sons Ltd.\n\nDOI: 10.1111/imr.12743 \nPMCID: PMC6464390 [Available on 2020-05-01]\nPMID: 30977196  [Indexed for MEDLINE]',
     '3. Genes Chromosomes Cancer. 2019 Sep;58(9):619-626. doi: 10.1002/gcc.22748. Epub\n2019 Apr 3.\n\nGATA3 germline variant is associated with CRLF2 expression and predicts outcome\nin pediatric B-cell precursor acute lymphoblastic leukemia.\n\nMadzio J(1)(2), Pastorczak A(1), Sedek L(3), Braun M(1)(2)(4), Taha J(1),\nWypyszczak K(1), Trelinska J(1), Lejman M(5), Muszynska-Roslan K(6), Tomasik\nB(1)(2)(7), Derwich K(8), Koltan A(9), Kazanowska B(10), Irga-Jaworska N(11),\nBadowska W(12), Matysiak M(13), Kowalczyk J(5), Styczynski J(9), Fendler\nW(7)(14), Szczepanski T(15), Mlynarski W(1).\n\nAuthor information: \n(1)Department of Pediatrics, Oncology and Hematology, Medical University of Lodz,\nLodz, Poland.\n(2)Postgraduate School of Molecular Medicine, Medical University of Warsaw,\nWarsaw, Poland.\n(3)Department of Microbiology and Immunology, Medical University of Silesia,\nKatowice, Poland.\n(4)Department of Pathology, Medical University of Lodz, Lodz, Poland.\n(5)Department of Pediatric Hematology and Oncology, Medical University of Lublin,\nLublin, Poland.\n(6)Department of Pediatric Oncology and Hematology, Medical University of\nBialystok, Bialystok, Poland.\n(7)Department of Biostatistics & Translational Medicine, Medical University of\nLodz, Lodz, Poland.\n(8)Department of Pediatric Hematology, Oncology and Transplantology, University\nof Medical Sciences, Poznan, Poland.\n(9)Department of Pediatric Hematology and Oncology, Collegium Medicum, Nicolaus\nCopernicus University, Torun, Poland.\n(10)Department of Pediatric Hematology, Oncology and Transplantology, Medical\nUniversity of Wroclaw, Wroclaw, Poland.\n(11)Department of Pediatric Hematology, Gdansk Medical University, Gdansk,\nPoland.\n(12)Department of Pediatric Hematology and Oncology, University of Warmia and\nMazury, Olsztyn, Poland.\n(13)Department of Pediatric Hematology and Oncology, Medical University of\nWarsaw, Warsaw, Poland.\n(14)Department of Radiation Oncology, Dana-Farber Cancer Institute, Boston,\nMassachusetts.\n(15)Department of Pediatric Hematology and Oncology, Zabrze Medical University of\nSilesia, Katowice, Poland.\n\nThe germline variant at rs3824662 in GATA3 is a risk locus for Philadelphia-like \nacute lymphoblastic leukemia (Ph-like ALL), the biological subtype of B-cell\nprecursor (BCP)-ALL defined by a distinct gene expression profile and the\npresence of specific somatic aberrations including rearrangements of CRLF2. In\nthis study, we investigated whether rs3824662 in GATA3 associates with CRLF2\nexpression in leukemic cells and predicts prognosis in pediatric BCP-ALL patients\ntreated according to the ALL Intercontinental Berlin-Frankfurt-Münster (IC BFM)\n2009 (n\u2009=\u2009645) and the ALL IC BFM 2002 (n\u2009=\u2009216) protocols. High expression of\nCRLF2 was observed at both protein and mRNA levels (fourfold higher in AA than in\nCA\u2009+\u2009CC) among GATA3 AA variant carriers, independent of the presence of\nP2RY8-CRLF2 fusion. Additionally, the AA variant at rs3824662 was a significant\nfactor affecting minimal residual disease level at the end of induction phase and\noverall survival regardless of the risk group and the protocol. The germline\nvariant at rs3824662 in GATA3 is a prognostic factor which associates with CRLF2 \nexpression in leukemic cells supporting the hypothesis that GATA3 may have a\nregulatory effect on the CRLF2 pathway in pediatric BCP-ALL.\n\n© 2019 Wiley Periodicals, Inc.\n\nDOI: 10.1002/gcc.22748 \nPMID: 30859636  [Indexed for MEDLINE]',
     '4. Nature. 2019 Mar;567(7747):244-248. doi: 10.1038/s41586-019-1003-z. Epub 2019 Mar\n6.\n\nS-Geranylgeranyl-L-glutathione is a ligand for human B cell-confinement receptor \nP2RY8.\n\nLu E(1), Wolfreys FD(1), Muppidi JR(1)(2), Xu Y(1), Cyster JG(3).\n\nAuthor information: \n(1)Howard Hughes Medical Institute and Department of Microbiology and Immunology,\nUniversity of California, San Francisco, CA, USA.\n(2)Lymphoid Malignancies Branch, National Cancer Institute, National Institutes\nof Health, Bethesda, MD, USA.\n(3)Howard Hughes Medical Institute and Department of Microbiology and Immunology,\nUniversity of California, San Francisco, CA, USA. jason.cyster@ucsf.edu.\n\nGerminal centres are important sites for antibody diversification and affinity\nmaturation, and are also a common origin of B cell malignancies. Despite being\nmade up of motile cells, germinal centres are tightly confined within B cell\nfollicles. The cues that promote this confinement are incompletely understood.\nP2RY8 is a Gα13-coupled receptor that mediates the inhibition of migration and\nregulates the growth of B cells in lymphoid tissues1,2. P2RY8 is frequently\nmutated in germinal-centre B cell-like diffuse large B cell lymphoma (GCB-DLBCL) \nand Burkitt lymphoma1,3-6, and the ligand for this receptor has not yet been\nidentified. Here we perform a search for P2RY8 ligands and find P2RY8 bioactivity\nin bile and in culture supernatants of several mouse and human cell lines. Using \na seven-step biochemical fractionation procedure and a drop-out mass spectrometry\napproach, we show that a previously undescribed biomolecule,\nS-geranylgeranyl-L-glutathione (GGG), is a potent P2RY8 ligand that is detectable\nin lymphoid tissues at the nanomolar level. GGG inhibited the chemokine-mediated \nmigration of human germinal-centre B cells and T follicular helper cells, and\nantagonized the induction of phosphorylated AKT in germinal-centre B cells. We\nalso found that the enzyme gamma-glutamyltransferase-5 (GGT5), which was highly\nexpressed by follicular dendritic cells, metabolized GGG to a form that did not\nactivate the receptor. Overexpression of GGT5 disrupted the ability of P2RY8 to\npromote B cell confinement to germinal centres, which\xa0indicates that GGT5\nestablishes a GGG gradient in lymphoid tissues. This work defines GGG as an\nintercellular signalling molecule\xa0that is involved in organizing and controlling \ngerminal-centre responses. As the P2RY8 locus is modified in several other types \nof\xa0cancer in addition to GCB-DLBCL and Burkitt lymphoma, we speculate that GGG\nmight have organizing and growth-regulatory roles in multiple human tissues.\n\nDOI: 10.1038/s41586-019-1003-z \nPMCID: PMC6640153\nPMID: 30842656  [Indexed for MEDLINE]',
     "5. Leukemia. 2019 Apr;33(4):893-904. doi: 10.1038/s41375-018-0297-4. Epub 2018 Nov\n28.\n\nSingle-cell analysis identifies CRLF2 rearrangements as both early and late\nevents in Down syndrome and non-Down syndrome acute lymphoblastic leukaemia.\n\nPotter N(1), Jones L(2), Blair H(2), Strehl S(3), Harrison CJ(2), Greaves M(1),\nKearney L(1), Russell LJ(4).\n\nAuthor information: \n(1)The Institute of Cancer Research, London, UK.\n(2)Northern Institute for Cancer Research, Newcastle University,\nNewcastle-upon-Tyne, UK.\n(3)CCRI, Children's Cancer Research Institute, St. Anna Kinderkrebsforschung,\nVienna, Austria.\n(4)Northern Institute for Cancer Research, Newcastle University,\nNewcastle-upon-Tyne, UK. lisa.russell@newcastle.ac.uk.\n\nDeregulated expression of the type I cytokine receptor, CRLF2, is observed in\n5-15% of precursor B-cell acute lymphoblastic leukaemia (B-ALL). We have\npreviously reported the genomic landscape of patients with CRLF2 rearrangements\n(CRLF2-r) using both whole genome and exome sequencing, which identified a number\nof potential clonal and sub-clonal genomic alterations. In this study, we aimed\nto assess when the CRLF2-r; IGH-CRLF2 or P2RY8-CRLF2, arose during the evolution \nof both Down syndrome-ALL (DS-ALL) and non-DS-ALL. Using fluorescence in situ\nhybridisation, we were able to track up to four structural variants in single\ncells from 47 CRLF2-r B-ALL patients, which in association with our multiplex\nsingle-cell analysis of a further four patients, permitted simultaneous tracking \nof copy number alterations, structural and single nucleotide variants within\nindividual cells. We observed CRLF2-r arising as both early and late events in DS\nand non-DS-ALL patients. Parallel evolution of discrete clones was observed in\nthe development of CRLF2-r B-ALL, either involving the CRLF2-r or one of the\nother tracked abnormalities. In-depth single-cell analysis identified both linear\nand branching evolution with early clones harbouring a multitude of\nabnormalities, including the CRLF2-r in DS-ALL patients.\n\nDOI: 10.1038/s41375-018-0297-4 \nPMCID: PMC6398588\nPMID: 30487598  [Indexed for MEDLINE]",
     "6. Gigascience. 2018 Jul 1;7(7). doi: 10.1093/gigascience/giy079.\n\nClinker: visualizing fusion genes detected in RNA-seq data.\n\nSchmidt BM(1), Davidson NM(1)(2), Hawkins ADK(1), Bartolo R(1), Majewski\nIJ(3)(4), Ekert PG(1), Oshlack A(1)(2).\n\nAuthor information: \n(1)Murdoch Children's Research Institute, The Royal Children's Hospital,\nFlemington, Road, Parkville Vic 3052 Australia.\n(2)School of Biosciences, University of Melbourne, Parkivlle Vic 3010, Australia.\n(3)Division of Cancer and Haematology, The Walter and Eliza Hall Institute of\nMedical Research, Parkville Vic 3052, Australia.\n(4)Faculty of Medicine, Dentistry and Health Sciences, University of Melbourne,\nParkville Vic 3010, Australia.\n\nBackground: Genomic profiling efforts have revealed a rich diversity of oncogenic\nfusion genes. While there are many methods for identifying fusion genes from\nRNA-sequencing (RNA-seq) data, visualizing these transcripts and their supporting\nreads remains challenging.\nFindings: Clinker is a bioinformatics tool written in Python, R, and Bpipe that\nleverages the superTranscript method to visualize fusion genes. We demonstrate\nthe use of Clinker to obtain interpretable visualizations of the RNA-seq data\nthat lead to fusion calls. In addition, we use Clinker to explore multiple fusion\ntranscripts with novel breakpoints within the P2RY8-CRLF2 fusion gene in B-cell\nacute lymphoblastic leukemia.\nConclusions: Clinker is freely available software that allows visualization of\nfusion genes and the RNA-seq data used in their discovery.\n\nDOI: 10.1093/gigascience/giy079 \nPMCID: PMC6065480\nPMID: 29982439  [Indexed for MEDLINE]",
     "7. Int J Hematol. 2018 Sep;108(3):312-318. doi: 10.1007/s12185-018-2474-7. Epub 2018\nMay 21.\n\nCopy number abnormality of acute lymphoblastic leukemia cell lines based on their\ngenetic subtypes.\n\nTomoyasu C(1)(2), Imamura T(3), Tomii T(1), Yano M(4), Asai D(5), Goto H(6),\nShimada A(7), Sanada M(8), Iwamoto S(9), Takita J(10), Minegishi M(11), Inukai\nT(12), Sugita K(12), Hosoi H(1).\n\nAuthor information: \n(1)Department of Pediatrics, Graduate School of Medical Science, Kyoto\nPrefectural University of Medicine, 465 Kajii-cho, Hirokoji, Kamigyo-ku, Kyoto,\n602-8566, Japan.\n(2)Department of Pediatrics, National Hospital Organization Maizuru Medical\nCenter, Maizuru, Japan.\n(3)Department of Pediatrics, Graduate School of Medical Science, Kyoto\nPrefectural University of Medicine, 465 Kajii-cho, Hirokoji, Kamigyo-ku, Kyoto,\n602-8566, Japan. imamura@koto.kpu-m.ac.jp.\n(4)Department of Pediatrics, Kyoto City Hospital, Kyoto, Japan.\n(5)Department of Pediatrics, Japanese Red Cross Kyoto Daini Hospital, Kyoto,\nJapan.\n(6)Division of Hemato-Oncology and Regenerative Medicine, Kanagawa Children's\nMedical Center, Yokohama, Japan.\n(7)Department of Pediatric Hematology/Oncology, Okayama University Hospital,\nOkayama, Japan.\n(8)Clinical Research Center, National Hospital Organization Nagoya Medical\nCenter, Nagoya, Japan.\n(9)Department of Pediatrics, Mie University Graduate School of Medicine, Tsu,\nJapan.\n(10)Department of Pediatrics, Graduate School of Medicine, The University of\nTokyo, Tokyo, Japan.\n(11)Japanese Red Cross Fukushima Blood Center, Okuma, Japan.\n(12)Department of Pediatrics, Yamanashi University, Kofu, Japan.\n\nIn this study, we performed genetic analysis of 83 B cell precursor acute\nlymphoblastic leukemia (B-ALL) cell lines. First, we performed multiplex\nligation-dependent probe amplification analysis to identify copy number\nabnormalities (CNAs) in eight genes associated with B-ALL according to genetic\nsubtype. In Ph+ B-ALL cell lines, the frequencies of IKZF1, CDKN2A/2B, BTG1, and \nPAX5 deletion were significantly higher than those in Ph- B-ALL cell lines. The\nfrequency of CDKN2A/2B deletion in KMT2A rearranged cell lines was significantly \nlower than that in non-KMT2A rearranged cell lines. These findings suggest that\nCNAs are correlated with genetic subtype in B-ALL cell lines. In addition, we\ndetermined that three B-other ALL cell lines had IKZF1 deletions (YCUB-5, KOPN49,\nand KOPN75); we therefore performed comprehensive genetic analysis of these cell \nlines. YCUB-5, KOPN49, and KOPN75 had P2RY8-CRLF2, IgH-CRLF2, and PAX5-ETV6\nfusions, respectively. Moreover, targeted capture sequencing revealed that YCUB-5\nhad JAK2 R683I and KRAS G12D, and KOPN49 had JAK2 R683G and KRAS G13D mutations. \nThese data may contribute to progress in the field of leukemia research.\n\nDOI: 10.1007/s12185-018-2474-7 \nPMID: 29786757  [Indexed for MEDLINE]",
     "8. Blood Adv. 2018 Mar 13;2(5):529-533. doi: 10.1182/bloodadvances.2017014704.\n\nPrognostic impact of kinase-activating fusions and IKZF1 deletions in pediatric\nhigh-risk B-lineage acute lymphoblastic leukemia.\n\nTran TH(1)(2), Harris MH(3), Nguyen JV(2), Blonquist TM(4), Stevenson KE(4),\nStonerock E(5), Asselin BL(6), Athale UH(7), Clavell LA(8), Cole PD(9), Kelly\nKM(10), Laverdiere C(1), Leclerc JM(1), Michon B(11), Schorin MA(12), Welch\nJJG(13), Reshmi SC(5), Neuberg DS(4), Sallan SE(14), Loh ML(2), Silverman LB(14).\n\nAuthor information: \n(1)Division of Pediatric Hematology-Oncology, Charles-Bruneau Cancer Center,\nCentre Hospitalier Universitaire Sainte-Justine Research Center, Montréal, QC,\nCanada.\n(2)Helen Diller Family Cancer Research Center, Department of Pediatrics, Benioff \nChildren's Hospital, University of California, San Francisco, San Francisco, CA.\n(3)Department of Pathology, Boston Children's Hospital, Boston, MA.\n(4)Department of Biostatistics and Computational Biology, Dana-Farber Cancer\nInstitute, Boston, MA.\n(5)Department of Pathology and Laboratory Medicine, Nationwide Children's\nHospital, Columbus, OH.\n(6)Department of Pediatrics, University of Rochester School of Medicine and\nWilmot Cancer Institute, Rochester, NY.\n(7)Division of Hematology/Oncology, McMaster Children's Hospital, Hamilton Health\nSciences, Hamilton, ON, Canada.\n(8)San Jorge Children's Hospital, San Juan, Puerto Rico.\n(9)Department of Pediatrics, Montefiore Medical Center, Albert Einstein College\nof Medicine, Bronx, NY.\n(10)Division of Pediatric Hematology/Oncology/Stem Cell Transplantation, Columbia\nUniversity Medical Center, New York, NY.\n(11)Département de Pédiatrie, Centre Mère-Enfant Soleil, Centre Hospitalier\nUniversitaire de l'Université Laval, Québec City, QC, Canada.\n(12)Inova Fairfax Hospital for Children, Falls Church, VA.\n(13)Division of Pediatric Hematology/Oncology, Hasbro Children's Hospital/Brown\nUniversity, Providence, RI; and.\n(14)Department of Pediatric Oncology, Dana-Farber Cancer Institute and Boston\nChildren's Hospital, Boston, MA.\n\nRecurrent chromosomal rearrangements carry prognostic significance in pediatric\nB-lineage acute lymphoblastic leukemia (B-ALL). Recent genome-wide analyses\nidentified a high-risk B-ALL subtype characterized by a diverse spectrum of\ngenetic alterations activating kinases and cytokine receptor genes. This subtype \nis associated with a poor prognosis when treated with conventional chemotherapy\nbut has demonstrated sensitivity to the relevant tyrosine kinase inhibitors. We\nsought to determine the frequency of kinase-activating fusions among National\nCancer Institute (NCI) high-risk, Ph-negative, B-ALL patients enrolled on\nDana-Farber Cancer Institute ALL Consortium Protocol 05-001 and to describe their\nassociated clinical characteristics and outcomes. Among the 105 patients\nscreened, 16 (15%) harbored an ABL-class fusion (ETV6-ABL1: n = 1; FOXP1-ABL1: n \n= 1; SFPQ-ABL1: n = 1; ZC3HAV1-ABL2: n = 1) or a fusion activating the JAK-STAT\npathway (P2RY8-CRLF2: n = 8; PAX5-JAK2: n = 4). Sixty-nine percent of patients\nwith an identified fusion had a concomitant IKZF1 deletion (n = 11). In\nunivariate analysis, fusion-positivity and IKZF1 deletion were each associated\nwith inferior event-free survival; IKZF1 deletion retained statistical\nsignificance in multivariable analysis (hazard ratio, 2.64; P = .019). Our\nfindings support therapy intensification for IKZF1-altered patients, irrespective\nof the presence of a kinase-activating fusion.\n\n© 2018 by The American Society of Hematology.\n\nDOI: 10.1182/bloodadvances.2017014704 \nPMCID: PMC5851421\nPMID: 29507076  [Indexed for MEDLINE]",
     '9. Leuk Res. 2018 Mar;66:79-84. doi: 10.1016/j.leukres.2018.01.012. Epub 2018 Feb 3.\n\nHigh frequency of intermediate and poor risk copy number abnormalities in\npediatric cohort of B-ALL correlate with high MRD post induction.\n\nSingh M(1), Bhatia P(2), Trehan A(1), Varma N(3), Sachdeva MS(3), Bansal D(1),\nJain R(1), Naseem S(3).\n\nAuthor information: \n(1)Paediatric Haematology-Oncology Unit, Post Graduate Institute of Medical\nEducation and Research, Chandigarh, India.\n(2)Paediatric Haematology-Oncology Unit, Post Graduate Institute of Medical\nEducation and Research, Chandigarh, India. Electronic address:\nbhatia.prateek@pgimer.edu.in.\n(3)Department of Haematology, Post Graduate Institute of Medical Education and\nResearch, Chandigarh, India.\n\nCopy number abnormalities (CNAs) and recurrent fusion transcripts are important\ngenetic events which define and prognosticate B-Cell Acute Lymphoblastic Leukemia\n(B-ALL). We evaluated CNAs and fusion transcripts in 67 pediatric B-ALL cases and\ncorrelated the data with standard risk factors and early treatment outcome\nparameters. Common fusion transcripts ETV6-RUNX1, E2A-PBX, BCR-ABL1 and MLL-AF4\nwere examined by RT-PCR and noted in 15%, 15%, 13% and 1.4% of all cases\nrespectively. CNAs in IKZF1, PAX5, EBF1, BTG1, RB1, CDKN2A/B and genes from PAR1 \nregion viz., CSF2RA, IL3RA,P2RY8, SHOX region and CRLF2 were analyzed by\nmultiplex ligation dependent probe amplification assay and were detected in 70%\n(47/67) of cases, with predominantly deletions in CDKN2A/B (36%), PAX5 (18%) and \nIKZF1 (16%). A statistically significant association of intermediate/poor risk\nCNAs was noted with high WBC count (p\u202f=\u202f0.001), NCI group (p\u202f=\u202f0.02) and minimal \nresidual disease at Day35 (p\u202f<\u202f0.0001). IKZF1 and CDKN2A/B deletion revealed poor\nEFS of 56% at 24 months as compared to EFS of 80% in rest of the cases (p\u202f=\u202f0.05)\nsuggesting their potential role in early risk stratification.\n\nCopyright © 2018 Elsevier Ltd. All rights reserved.\n\nDOI: 10.1016/j.leukres.2018.01.012 \nPMID: 29407587  [Indexed for MEDLINE]',
     "10. Oncotarget. 2017 Oct 26;8(57):96466-96467. doi: 10.18632/oncotarget.22098.\neCollection 2017 Nov 14.\n\nThe enigmatic role(s) of P2RY8-CRLF2.\n\nPanzer-Grümayer R(1), Köhrer S(1), Haas OA(1).\n\nAuthor information: \n(1)Renate Panzer-Grümayer: Leukemia Biology Group, Children's Cancer Research\nInstitute, Vienna, Austria.\n\nDOI: 10.18632/oncotarget.22098 \nPMCID: PMC5722491\nPMID: 29228539 ",
     "11. Br J Haematol. 2018 Feb;180(4):550-562. doi: 10.1111/bjh.15056. Epub 2017 Nov 30.\n\nA risk score including microdeletions improves relapse prediction for standard\nand medium risk precursor B-cell acute lymphoblastic leukaemia in children.\n\nSutton R(1)(2), Venn NC(1), Law T(1), Boer JM(3)(4), Trahair TN(1)(2)(5), Ng\nA(6), Den Boer ML(3)(7), Dissanayake A(1), Giles JE(1), Dalzell P(8), Mayoh C(1),\nBarbaric D(5), Revesz T(9), Alvaro F(10), Pieters R(4)(7), Haber M(1)(2), Norris \nMD(1)(11), Schrappe M(12), Dalla Pozza L(6), Marshall GM(1)(2)(5).\n\nAuthor information: \n(1)Children's Cancer Institute, Lowy Cancer Research Centre, UNSW, Sydney,\nAustralia.\n(2)School of Women's and Children's Health, UNSW, Sydney, Australia.\n(3)Department of Paediatric Oncology/Haematology, Erasmus Medical Centre, Sophia \nChildren's Hospital, Rotterdam, The Netherlands.\n(4)Princess Máxima Centre for Paediatric Oncology, Utrecht, The Netherlands.\n(5)Kids Cancer Centre, Sydney Children's Hospital, Randwick, Australia.\n(6)Cancer Centre for Children, The Children's Hospital at Westmead, Westmead,\nAustralia.\n(7)Dutch Childhood Oncology Group, The Hague, The Netherlands.\n(8)SEALS Genetics, NSW Health Pathology, Randwick, Australia.\n(9)Women's and Children's Hospital, SA Pathology, University of Adelaide,\nAdelaide, Australia.\n(10)John Hunter Children's Hospital, Newcastle, Australia.\n(11)UNSW Centre for Childhood Cancer Research, Kensington, Australia.\n(12)Christian-Albrechts-University Kiel and University Medical Centre\nSchleswig-Holstein, Kiel, Germany.\n\nTo prevent relapse, high risk paediatric acute lymphoblastic leukaemia (ALL) is\ntreated very intensively. However, most patients who eventually relapse have\nstandard or medium risk ALL with low minimal residual disease (MRD) levels. We\nanalysed recurrent microdeletions and other clinical prognostic factors in a\ncohort of 475 uniformly treated non-high risk precursor B-cell ALL patients with \nthe aim of better predicting relapse and refining risk stratification. Lower\nrelapse-free survival at 7\xa0years (RFS) was associated with IKZF1 intragenic\ndeletions (P\xa0<\xa00·0001); P2RY8-CRLF2 gene fusion (P\xa0<\xa00·0004); Day 33 MRD>5\xa0×\xa010-5\n(P\xa0<\xa00·0001) and High National Cancer Institute (NCI) risk (P\xa0<\xa00·0001). We\ncreated a predictive model based on a risk score (RS) for deletions, MRD and NCI \nrisk, extending from an RS of 0 (RS0) for patients with no unfavourable factors\nto RS2\xa0+\xa0 for patients with 2 or 3 high risk factors. RS0, RS1, and RS2\xa0+\xa0 groups\nhad RFS of 93%, 78% and 49%, respectively, and overall survival (OS) of 99%, 91% \nand 71%. The RS provided greater discrimination than MRD-based risk\nstratification into standard (89% RFS, 96% OS) and medium risk groups (79% RFS,\n91% OS). We conclude that this RS may enable better early therapeutic\nstratification and thus improve cure rates for childhood ALL.\n\n© 2017 John Wiley & Sons Ltd.\n\nDOI: 10.1111/bjh.15056 \nPMID: 29194562  [Indexed for MEDLINE]",
     '12. Am J Clin Pathol. 2017 Nov 20;148(6):523-528. doi: 10.1093/ajcp/aqx111.\n\nPhiladelphia Chromosome-like Mixed-Phenotype Acute Leukemia Demonstrating\nP2RY8-CRLF2 Fusion and JAK1 Mutation.\n\nChoi SM(1), Frederiksen JK(1), Ross CW(1), Bixby DL(2), Shao L(1).\n\nAuthor information: \n(1)Departments of Pathology.\n(2)Internal Medicine, University of Michigan, Ann Arbor.\n\nObjectives: Philadelphia chromosome-like (Ph-like) genetic alterations define a\nsubset of B lymphoblastic leukemia/lymphoma (B-ALL), which represents a separate \nprovisional entity in the World Health Organization 2016 updated classification. \nHowever, these alterations have not been described outside the context of B-ALL.\nMethods: Cytogenomic array and molecular analysis identified a Ph-like signature \nin a mixed-phenotype acute leukemia (MPAL), B/myeloid, confirmed using\nconventional immunophenotypic and cytochemical analysis.\nResults: Flow cytometry identified a blast population demonstrating a B-cell\nlineage and myeloperoxidase positivity. A P2RY8-CRLF2 fusion and JAK1 mutation\nwere detected, both of which are associated with Ph-like features.\nConclusions: To our knowledge, this is the first report of Ph-like MPAL, which\nmay represent a new diagnostic entity. We emphasize the need for refinement of\ndiagnostic criteria for MPALs and highlight an opportunity for expansion of\ninclusion criteria in ongoing clinical trials studying the use of tyrosine kinase\ninhibitor therapy to include cases of Ph-like MPAL.\n\n© American Society for Clinical Pathology, 2017. All rights reserved. For\npermissions, please e-mail: journals.permissions@oup.com\n\nDOI: 10.1093/ajcp/aqx111 \nPMID: 29140408  [Indexed for MEDLINE]',
     "13. Cancer Lett. 2017 Nov 1;408:92-101. doi: 10.1016/j.canlet.2017.08.034. Epub 2017 \nSep 1.\n\nDifferential expression of MUC4, GPR110 and IL2RA defines two groups of\nCRLF2-rearranged acute lymphoblastic leukemia patients with distinct secondary\nlesions.\n\nSadras T(1), Heatley SL(1), Kok CH(1), Dang P(2), Galbraith KM(2), McClure BJ(2),\nMuskovic W(3), Venn NC(3), Moore S(4), Osborn M(5), Revesz T(6), Moore AS(7),\nHughes TP(8), Yeung D(9), Sutton R(10), White DL(11).\n\nAuthor information: \n(1)Cancer Theme, South Australian Health & Medical Research Institute, Adelaide, \nSA, Australia; Discipline of Medicine, University of Adelaide, Adelaide, SA,\nAustralia.\n(2)Cancer Theme, South Australian Health & Medical Research Institute, Adelaide, \nSA, Australia.\n(3)Molecular Diagnostics Program, Children's Cancer Institute, Lowy Cancer\nResearch Centre, UNSW Sydney, NSW, Australia.\n(4)Department of Genetic Pathology, SA Pathology, Adelaide, SA, Australia.\n(5)SA Pathology at Women's & Children's Hospital, Adelaide, SA, Australia;\nAustralian Genomic Health Alliance, Adelaide, SA, Australia.\n(6)SA Pathology at Women's & Children's Hospital, Adelaide, SA, Australia.\n(7)The University of Queensland Diamantina Institute, UQ Child Health Research\nCentre, The University of Queensland, Brisbane, QLD, Australia; Oncology Services\nGroup, Children's Health Queensland Hospital and Health Service, Brisbane, QLD,\nAustralia.\n(8)Cancer Theme, South Australian Health & Medical Research Institute, Adelaide, \nSA, Australia; Discipline of Medicine, University of Adelaide, Adelaide, SA,\nAustralia; Department of Haematology, SA Pathology, Adelaide, SA, Australia.\n(9)Department of Haematology, SA Pathology, Adelaide, SA, Australia.\n(10)Molecular Diagnostics Program, Children's Cancer Institute, Lowy Cancer\nResearch Centre, UNSW Sydney, NSW, Australia; Australian Genomic Health Alliance,\nAdelaide, SA, Australia; School of Women's and Children's Health, Medicine,\nUniversity of NSW, Sydney, NSW, Australia.\n(11)Cancer Theme, South Australian Health & Medical Research Institute, Adelaide,\nSA, Australia; Discipline of Medicine, University of Adelaide, Adelaide, SA,\nAustralia; Australian Genomic Health Alliance, Adelaide, SA, Australia.\nElectronic address: deborah.white@sahmri.com.\n\nCRLF2-rearrangements (CRLF2-r) occur frequently in Ph-like B-ALL, a high-risk ALL\nsub-type characterized by a signaling profile similar to Ph\xa0+\xa0ALL, however\naccumulating evidence indicates genetic heterogeneity within CRLF2-r ALL. We\nperformed thorough genomic characterization of 35 CRLF2-r cases (P2RY8-CRLF2\nn\xa0=\xa018; IGH-CRLF2 n\xa0=\xa017). Activating JAK2 mutations were present in 34% of\npatients, and a CRLF2-F232C mutation was identified in an additional 17%. IKZF1\ndeletions were detected in 63% of cases. The majority of patients (26/35)\nclassified as Ph-like, and these were characterized by significantly higher\nlevels of MUC4, GPR110 and IL2RA/CD25. In addition, Ph-like CRLF2-r samples were \nsignificantly enriched for IKZF1 deletions, JAK2/CRLF2 mutations and increased\nexpression of JAK/STAT target genes (CISH, SOCS1), suggesting that\nmutation-driven CRLF2/JAK2 activation is more frequent in this sub-group. Less is\nknown about the genomics of CRLF2-r cases lacking JAK2-pathway mutations, but\nKRAS/NRAS mutations were identified in 4/9 non-Ph-like samples. This work\nhighlights the heterogeneity of secondary lesions which may arise and influence\nintracellular-pathway activation in CRLF2-r patients, and importantly presents\ndistinct therapeutic targets within a group of patients harboring identical\nprimary translocations, for whom efficient directed therapies are currently\nlacking.\n\nCopyright © 2017 Elsevier B.V. All rights reserved.\n\nDOI: 10.1016/j.canlet.2017.08.034 \nPMID: 28866095  [Indexed for MEDLINE]",
     "14. Genes Chromosomes Cancer. 2017 Oct;56(10):730-749. doi: 10.1002/gcc.22477. Epub\n2017 Jul 1.\n\nCopy number alterations determined by single nucleotide polymorphism array\ntesting in the clinical laboratory are indicative of gene fusions in pediatric\ncancer patients.\n\nBusse TM(1), Roth JJ(2), Wilmoth D(3), Wainwright L(3), Tooke L(3), Biegel\nJA(1)(4).\n\nAuthor information: \n(1)Department of Pathology and Laboratory Medicine Children's Hospital of Los\nAngeles, Center for Personalized Medicine, Los Angeles, California.\n(2)Department of Pathology and Laboratory Medicine, Hospital of the University of\nPennsylvania, Philadelphia, Pennsylvania.\n(3)Department of Pathology and Laboratory Medicine, The Children's Hospital of\nPhiladelphia, Philadelphia, Pennsylvania.\n(4)Department of Pathology, USC Keck School of Medicine, Los Angeles, California.\n\nGene fusions resulting from structural rearrangements are an established\nmechanism of tumorigenesis in pediatric cancer. In this clinical cohort, 1,350\nsingle nucleotide polymorphism (SNP)-based chromosomal microarrays from 1,211\npediatric cancer patients were evaluated for copy number alterations (CNAs)\nassociated with gene fusions. Karyotype or fluorescence in situ hybridization\nstudies were performed in 42% of the patients. Ten percent of the bone marrow or \nsolid tumor specimens had SNP array-associated CNAs suggestive of a gene fusion. \nAlterations involving ETV6, ABL1-NUP214, EBF1-PDGFRB, KMT2A(MLL), LMO2-RAG,\nMYH11-CBFB, NSD1-NUP98, PBX1, STIL-TAL1, ZNF384-TCF3, P2RY8-CRLF2, and\nRUNX1T1-RUNX1 fusions were detected in the bone marrow samples. The most common\nalteration among the low-grade gliomas was a 7q34 tandem duplication resulting in\na KIAA1549-BRAF fusion. Additional fusions identified in the pediatric brain\ntumors included FAM131B-BRAF and RAF1-QKI. COL1A1-PDGFB, CRTC1-MAML2, EWSR1,\nHEY1, PAX3- and PAX7-FOXO1, and PLAG1 fusions were determined in a variety of\nsolid tumors and a novel potential gene fusion, FGFR1-USP6, was detected in an\naneurysmal bone cyst. The identification of these gene fusions was instrumental\nin tumor diagnosis. In contrast to hematologic and solid tumors in adults that\nare predominantly driven by mutations, the majority of hematologic and solid\ntumors in children are characterized by CNAs and gene fusions. Chromosomal\nmicroarray analysis is therefore a robust platform to identify diagnostic and\nprognostic markers in the clinical setting.\n\n© 2017 Wiley Periodicals, Inc.\n\nDOI: 10.1002/gcc.22477 \nPMID: 28597942  [Indexed for MEDLINE]",
     "15. Blood. 2017 Jun 22;129(25):3352-3361. doi: 10.1182/blood-2016-12-758979. Epub\n2017 Apr 13.\n\nTargetable kinase gene fusions in high-risk B-ALL: a study from the Children's\nOncology Group.\n\nReshmi SC(1)(2), Harvey RC(3), Roberts KG(4), Stonerock E(1), Smith A(5), Jenkins\nH(1), Chen IM(3), Valentine M(6), Liu Y(7), Li Y(7), Shao Y(4), Easton J(7),\nPayne-Turner D(4), Gu Z(4), Tran TH(8), Nguyen JV(8), Devidas M(9), Dai Y(9),\nHeerema NA(2), Carroll AJ 3rd(10), Raetz EA(11), Borowitz MJ(12), Wood BL(13),\nAngiolillo AL(14), Burke MJ(15), Salzer WL(16), Zweidler-McKay PA(17), Rabin\nKR(18), Carroll WL(19), Zhang J(7), Loh ML(8), Mullighan CG(4), Willman CL(3),\nGastier-Foster JM(1)(2)(20), Hunger SP(21).\n\nAuthor information: \n(1)Institute for Genomic Medicine, Nationwide Children's Hospital, Columbus, OH.\n(2)Department of Pathology, The Ohio State University College of Medicine,\nColumbus, OH.\n(3)University of New Mexico Cancer Center, Albuquerque, NM.\n(4)Department of Pathology, St. Jude Children's Research Hospital, Memphis, TN.\n(5)Comprehensive Cancer Center, The Ohio State University College of Medicine,\nColumbus, OH.\n(6)Department of Cytogenetics and.\n(7)Department of Computational Biology, St. Jude Children's Research Hospital,\nMemphis, TN.\n(8)Department of Pediatrics, UCSF Benioff Children's Hospital and Helen Diller\nFamily Comprehensive Cancer Center, University of California, San Francisco, San \nFrancisco, CA.\n(9)Department of Biostatistics, University of Florida, Gainesville, FL.\n(10)Department of Genetics, University of Alabama at Birmingham, Birmingham, AL.\n(11)Department of Pediatrics, University of Utah, Salt Lake City, UT.\n(12)Department of Pathology, Johns Hopkins University, Baltimore, MD.\n(13)Seattle Children's Hospital, Seattle, WA.\n(14)Children's National Medical Center, Washington, DC.\n(15)Department of Pediatrics, Medical College of Wisconsin, Milwaukee, WI.\n(16)US Army Medical Research and Materiel Command, Fort Detrick, MD.\n(17)Department of Pediatrics, MD Anderson Cancer Center, Houston, TX.\n(18)Department of Pediatrics, Baylor College of Medicine, Houston, TX.\n(19)Perlmutter Cancer Center, NYU Langone Medical Center, New York, NY.\n(20)Department of Pediatrics, The Ohio State University College of Medicine,\nColumbus, OH; and.\n(21)Department of Pediatrics, Children's Hospital of Philadelphia and Perelman\nSchool of Medicine, University of Pennsylvania, Philadelphia, PA.\n\nComment in\n    Blood. 2017 Jun 22;129(25):3280-3282.\n\nPhiladelphia chromosome-like (Ph-like) acute lymphoblastic leukemia (ALL) is a\nhigh-risk subtype characterized by genomic alterations that activate cytokine\nreceptor and kinase signaling. We examined the frequency and spectrum of\ntargetable genetic lesions in a retrospective cohort of 1389 consecutively\ndiagnosed patients with childhood B-lineage ALL with high-risk clinical features \nand/or elevated minimal residual disease at the end of remission induction\ntherapy. The Ph-like gene expression profile was identified in 341 of 1389\npatients, 57 of whom were excluded from additional analyses because of the\npresence of BCR-ABL1 (n = 46) or ETV6-RUNX1 (n = 11). Among the remaining 284\npatients (20.4%), overexpression and rearrangement of CRLF2 (IGH-CRLF2 or\nP2RY8-CRLF2) were identified in 124 (43.7%), with concomitant genomic alterations\nactivating the JAK-STAT pathway (JAK1, JAK2, IL7R) identified in 63 patients\n(50.8% of those with CRLF2 rearrangement). Among the remaining patients, using\nreverse transcriptase polymerase chain reaction or transcriptome sequencing, we\nidentified targetable ABL-class fusions (ABL1, ABL2, CSF1R, and PDGFRB) in 14.1%,\nEPOR rearrangements or JAK2 fusions in 8.8%, alterations activating other\nJAK-STAT signaling genes (IL7R, SH2B3, JAK1) in 6.3% or other kinases (FLT3,\nNTRK3, LYN) in 4.6%, and mutations involving the Ras pathway (KRAS, NRAS, NF1,\nPTPN11) in 6% of those with Ph-like ALL. We identified 8 new rearrangement\npartners for 4 kinase genes previously reported to be rearranged in Ph-like ALL. \nThe current findings provide support for the precision-medicine testing and\ntreatment approach for Ph-like ALL implemented in Children's Oncology Group ALL\ntrials.\n\nDOI: 10.1182/blood-2016-12-758979 \nPMCID: PMC5482101\nPMID: 28408464  [Indexed for MEDLINE]",
     '16. Genes Chromosomes Cancer. 2017 Aug;56(8):608-616. doi: 10.1002/gcc.22464. Epub\n2017 May 5.\n\nETV6/RUNX1-like acute lymphoblastic leukemia: A novel B-cell precursor leukemia\nsubtype associated with the CD27/CD44 immunophenotype.\n\nZaliova M(1)(2)(3), Kotrova M(1)(2)(3), Bresolin S(4), Stuchly J(1)(2), Stary\nJ(2)(3), Hrusak O(1)(2)(3), Te Kronnie G(4), Trka J(1)(2)(3), Zuna J(1)(2)(3),\nVaskova M(1)(2).\n\nAuthor information: \n(1)CLIP-Childhood Leukaemia Investigation Prague, Czech Republic.\n(2)Department of Paediatric Haematology and Oncology, Second Faculty of Medicine,\nCharles University, Prague, Czech Republic.\n(3)University Hospital Motol, Prague, Czech Republic.\n(4)Department of Women\'s and Children\'s Health, University of Padova, Italy.\n\nWe have shown previously that ETV6/RUNX1-positive acute lymphoblastic leukemia\n(ALL) is distinguishable from other ALL subtypes by CD27pos /CD44low-neg\nimmunophenotype. During diagnostic immunophenotyping of 573 childhood B-cell\nprecursor ALL (BCP-ALL), we identified eight cases with this immunophenotype\namong "B-other ALL" (BCP-ALL cases negative for routinely tested\nchromosomal/genetic aberrations). We aimed to elucidate whether these cases\nbelong to the recently described ETV6/RUNX1-like ALL defined by the\nETV6/RUNX1-specific gene expression profile (GEP), harboring concurrent ETV6 and \nIKZF1 lesions. We performed comprehensive genomic analysis using single\nnucleotide polymorphism arrays, whole exome and transcriptome sequencing and GEP \non microarrays. In unsupervised hierarchical clustering based on GEP, five out of\nseven analyzed CD27pos /CD44low-neg B-other cases clustered with\nETV6/RUNX1-positive ALL and were thus classified as ETV6/RUNX1-like ALL. The two \ncases clustering outside ETV6/RUNX1-positive ALL harbored a P2RY8/CRLF2 fusion\nwith activating JAK2 mutations and a TCF3/ZNF384 fusion, respectively, assigning \nthem to other ALL subtypes. All five ETV6/RUNX1-like cases harbored ETV6\ndeletions; uniform intragenic ARPP21 deletions and various IKZF1 lesions were\neach found in three ETV6/RUNX1-like cases. The frequency of ETV6 and ARPP21\ndeletions was significantly higher in ETV6/RUNX1-like ALL compared with a\nreference cohort of 42 B-other ALL. In conclusion, we show that ETV6/RUNX1-like\nALL is associated with CD27pos /CD44low-neg immunophenotype and identify ARPP21\ndeletions to contribute to its specific genomic profile enriched for ETV6 and\nIKZF1 lesions. In conjunction with previously published data, our study\nidentifies the ETV6 lesion as the only common genetic aberration and thus the\nmost likely key driver of ETV6/RUNX1-like ALL.\n\n© 2017 Wiley Periodicals, Inc.\n\nDOI: 10.1002/gcc.22464 \nPMID: 28395118  [Indexed for MEDLINE]',
     "17. Pediatr Blood Cancer. 2017 Oct;64(10). doi: 10.1002/pbc.26539. Epub 2017 Apr 1.\n\nMolecular characterization of acute lymphoblastic leukemia with high CRLF2 gene\nexpression in childhood.\n\nSchmäh J(1), Fedders B(1), Panzer-Grümayer R(2), Fischer S(2), Zimmermann M(3),\nDagdan E(3), Bens S(4)(5), Schewe D(1), Moericke A(1), Alten J(1), Bleckmann\nK(1), Siebert R(4)(5), Schrappe M(1), Stanulla M(3), Cario G(1).\n\nAuthor information: \n(1)Department of Pediatrics, University Medical Center Schleswig-Holstein, Kiel, \nGermany.\n(2)Children's Cancer Research Institute, St. Anna Kinderkrebsforschung, Vienna,\nAustria.\n(3)Department of Pediatric Hematology and Oncology, Hannover Medical School,\nHannover, Germany.\n(4)Institute of Human Genetics, Christian-Albrechts-University Kiel & University \nMedical Center Schleswig-Holstein, Kiel, Germany.\n(5)Institute of Human Genetics, University of Ulm, Ulm, Germany.\n\nBACKGROUND: A high-level expression of the CRLF2 gene is frequent in precursor\nB-cell acute lymphoblastic leukemia (pB-ALL) and can be caused by different\ngenetic aberrations. The presence of the most frequent alteration, the\nP2RY8/CRLF2 fusion, was shown to be associated with a high relapse incidence in\nchildren treated according to ALL-Berlin-Frankfurt-Münster (BFM) protocols, which\nis poorly understood. Moreover, the frequency of other alterations has not been\nsystematically analyzed yet.\nPROCEDURE: CRLF2 mRNA expression and potential genetic aberrations causing a\nCRLF2 high expression were prospectively assessed in 1,105 patients treated\naccording to the Associazione Italiana Ematologia Oncologia Pediatrica\n(AIEOP)-BFM ALL 2009 protocol. Additionally, we determined copy number\nalterations in selected B-cell differentiation genes for all CRLF2\nhigh-expressing pB-ALL cases, as well as JAK2 and CRLF2 mutations.\nRESULTS: A CRLF2 high expression was detected in 26/178 (15%) T-cell acute\nlymphoblastic leukemia (T-ALL) cases, 21 of them (81%) had been stratified as\nhigh-risk patients by treatment response. In pB-ALL, a CRLF2 high expression was \ndetermined in 91/927 (10%) cases; the P2RY8/CRLF2 rearrangement in 44/91 (48%) of\nthem, supernumerary copies of CRLF2 in 18/91 (20%), and, notably, the IGH/CRLF2\ntranslocation was detected in 16/91 (18%). Remarkably, 7 of 16 (44%) patients\nwith IGH/CRLF2 translocation had already relapsed. P2RY8/CRLF2- and\nIGH/CRLF2-positive samples (70 and 94%, respectively) were characterized by a\nhigh frequency of additional deletions in B-cell differentiation genes such as\nIKZF1 or PAX5.\nCONCLUSION: Our data suggest that this high frequency of genetic aberrations in\nthe context of a high CRLF2 expression could contribute to the high risk of\nrelapse in P2RY8/CRLF2- and IGH/CRLF2-positive ALL.\n\n© 2017 Wiley Periodicals, Inc.\n\nDOI: 10.1002/pbc.26539 \nPMID: 28371317  [Indexed for MEDLINE]",
     "18. Genes Chromosomes Cancer. 2017 May;56(5):363-372. doi: 10.1002/gcc.22439. Epub\n2017 Jan 18.\n\nCharacterisation of the genomic landscape of CRLF2-rearranged acute lymphoblastic\nleukemia.\n\nRussell LJ(1), Jones L(1), Enshaei A(1), Tonin S(1), Ryan SL(1), Eswaran J(1),\nNakjang S(2), Papaemmanuil E(3)(4), Tubio JM(4), Fielding AK(5), Vora A(6),\nCampbell PJ(4), Moorman AV(1), Harrison CJ(1).\n\nAuthor information: \n(1)Leukaemia Research Cytogenetics Group, Northern Institute for Cancer Research,\nNewcastle University, Newcastle-upon-Tyne, UK.\n(2)Bioinformatics Support Unit, Newcastle University, Newcastle-upon-Tyne, UK.\n(3)Department of Epidemiology-Biostatistics, Memorial Sloan Kettering Cancer\nCenter, USA.\n(4)Cancer Genome Project, Wellcome Trust Sanger Institute, Hinxton, UK.\n(5)Research Department of Haemaoloty, UCL Cancer Institute, London, UK.\n(6)Department of Haematology, Sheffield Children's Hospital, Sheffield, UK.\n\nDeregulated expression of the type I cytokine receptor, CRLF2, is observed in\n5-15% of precursor B-cell acute lymphoblastic leukaemia (B-ALL). We aimed to\ndetermine the clinical and genetic landscape of those with IGH-CRLF2 or\nP2RY8-CRLF2 (CRLF2-r) using multiple genomic approaches. Clinical and demographic\nfeatures of CRLF2-r patients were characteristic of B-ALL. Patients with\nIGH-CRLF2 were older (14 y vs. 4 y, P\u2009<\u2009.001), while the incidence of CRLF2-r\namong Down syndrome patients was high (50/161, 31%). CRLF2-r co-occurred with\nprimary chromosomal rearrangements but the majority (111/161, 69%) had B-other\nALL. Copy number alteration (CNA) profiles were similar to B-other ALL, although \nCRLF2-r patients harbored higher frequencies of IKZF1 (60/138, 43% vs. 77/1351,\n24%) and BTG1 deletions (20/138, 15% vs. 3/1351, 1%). There were significant\ndifferences in CNA profiles between IGH-CRLF2 and P2RY8-CRLF2 patients: IKZF1\n(25/35, 71% vs. 36/108, 33%, P\u2009<\u2009.001), BTG1 (11/35, 31% vs. 10/108, 9%, P\n=.004), and ADD3 deletions (9/19, 47% vs. 5/38, 13%, P =.008). A novel gene\nfusion, USP9X-DDX3X, was discovered in 10/54 (19%) of patients. Pathway analysis \nof the mutational profile revealed novel involvement for focal adhesion. Although\nthe functional relevance of many of these abnormalities are unknown, they likely \nactivate additional pathways, which may represent novel therapeutic targets.\n\n© 2017 The Authors Genes, Chromosomes and Cancer Published byWiley Periodicals,\nInc.\n\nDOI: 10.1002/gcc.22439 \nPMCID: PMC5396319\nPMID: 28033648  [Indexed for MEDLINE]",
     "19. PLoS Med. 2016 Dec 13;13(12):e1002197. doi: 10.1371/journal.pmed.1002197.\neCollection 2016 Dec.\n\nHistological Transformation and Progression in Follicular Lymphoma: A Clonal\nEvolution Study.\n\nKridel R(1), Chan FC(1)(2), Mottok A(1)(3), Boyle M(1), Farinha P(1), Tan K(1),\nMeissner B(1), Bashashati A(4), McPherson A(4), Roth A(2)(4), Shumansky K(4), Yap\nD(4), Ben-Neriah S(1), Rosner J(4), Smith MA(2)(4), Nielsen C(4), Giné E(1),\nTelenius A(1), Ennishi D(1), Mungall A(5), Moore R(5), Morin RD(5)(6), Johnson\nNA(7), Sehn LH(1), Tousseyn T(8)(9), Dogan A(10)(11), Connors JM(1), Scott DW(1),\nSteidl C(1)(3), Marra MA(5), Gascoyne RD(1)(3), Shah SP(3)(4).\n\nAuthor information: \n(1)Centre for Lymphoid Cancer, BC Cancer Agency, Vancouver, British Columbia,\nCanada.\n(2)Bioinformatics Graduate Program, University of British Columbia, Vancouver,\nBritish Columbia, Canada.\n(3)Department of Pathology and Laboratory Medicine, University of British\nColumbia, Vancouver, British Columbia, Canada.\n(4)Department of Molecular Oncology, BC Cancer Agency, Vancouver, British\nColumbia, Canada.\n(5)Michael Smith Genome Sciences Centre, BC Cancer Agency, Vancouver, British\nColumbia, Canada.\n(6)Department of Molecular Biology and Biochemistry, Simon Fraser University,\nBurnaby, British Columbia, Canada.\n(7)Jewish General Hospital, Montreal, Quebec, Canada.\n(8)Translational Cell and Tissue Research Lab, Department for Imaging and\nPathology, University of Leuven (KU Leuven), Leuven, Belgium.\n(9)Department of Pathology, Universitaire Ziekenhuizen Leuven (UZ Leuven),\nLeuven, Belgium.\n(10)Department of Laboratory and Pathology, Mayo Clinic, Rochester, Minnesota,\nUnited States of America.\n(11)Departments of Pathology and Laboratory Medicine, Memorial Sloan Kettering\nCancer Center, New York, New York, United States of America.\n\nBACKGROUND: Follicular lymphoma (FL) is an indolent, yet incurable B cell\nmalignancy. A subset of patients experience an increased mortality rate driven by\ntwo distinct clinical end points: histological transformation and early\nprogression after immunochemotherapy. The nature of tumor clonal dynamics leading\nto these clinical end points is poorly understood, and previously determined\ngenetic alterations do not explain the majority of transformed cases or\naccurately predict early progressive disease. We contend that detailed knowledge \nof the expansion patterns of specific cell populations plus their associated\nmutations would provide insight into therapeutic strategies and disease biology\nover the time course of FL clinical histories.\nMETHODS AND FINDINGS: Using a combination of whole genome sequencing, targeted\ndeep sequencing, and digital droplet PCR on matched diagnostic and relapse\nspecimens, we deciphered the constituent clonal populations in 15 transformation \ncases and 6 progression cases, and measured the change in clonal population\nabundance over time. We observed widely divergent patterns of clonal dynamics in \ntransformed cases relative to progressed cases. Transformation specimens were\ngenerally composed of clones that were rare or absent in diagnostic specimens,\nconsistent with dramatic clonal expansions that came to dominate the\ntransformation specimens. This pattern was independent of time to transformation \nand treatment modality. By contrast, early progression specimens were composed of\nclones that were already present in the diagnostic specimens and exhibited only\nmoderate clonal dynamics, even in the presence of immunochemotherapy. Analysis of\nsomatic mutations impacting 94 genes was undertaken in an extension cohort\nconsisting of 395 samples from 277 patients in order to decipher disrupted\nbiology in the two clinical end points. We found 12 genes that were more commonly\nmutated in transformed samples than in the preceding FL tumors, including TP53,\nB2M, CCND3, GNA13, S1PR2, and P2RY8. Moreover, ten genes were more commonly\nmutated in diagnostic specimens of patients with early progression, including\nTP53, BTG1, MKI67, and XBP1.\nCONCLUSIONS: Our results illuminate contrasting modes of evolution shaping the\nclinical histories of transformation and progression. They have implications for \ninterpretation of evolutionary dynamics in the context of treatment-induced\nselective pressures, and indicate that transformation and progression will\nrequire different clinical management strategies.\n\nDOI: 10.1371/journal.pmed.1002197 \nPMCID: PMC5154502\nPMID: 27959929  [Indexed for MEDLINE]\n\nConflict of interest statement: I have read the journal's policy and the authors \nof this manuscript have the following competing interests: SPS is a founder and\nshareholder of Contextual Genomics Inc., developer of clinical genomic tests for \ncancer.",
     "20. Leukemia. 2017 Jul;31(7):1491-1501. doi: 10.1038/leu.2016.365. Epub 2016 Nov 30.\n\nGenomic and transcriptional landscape of P2RY8-CRLF2-positive childhood acute\nlymphoblastic leukemia.\n\nVesely C(1), Frech C(1), Eckert C(2), Cario G(3), Mecklenbräuker A(1), Zur Stadt \nU(4), Nebral K(1), Kraler F(1), Fischer S(1), Attarbaschi A(5), Schuster M(6),\nBock C(6), Cavé H(7), von Stackelberg A(2), Schrappe M(3), Horstmann MA(4), Mann \nG(5), Haas OA(1)(5), Panzer-Grümayer R(1).\n\nAuthor information: \n(1)Children's Cancer Research Institute (CCRI), Leukemia Biology Group, Vienna,\nAustria.\n(2)Department of Pediatric Oncology/Hematology, Charité Universitätsmedizin\nBerlin, Campus Virchow Klinikum, Berlin, Germany.\n(3)Department of Pediatrics, Medical University of Schleswig Holstein, Kiel,\nGermany.\n(4)Clinic of Pediatric Hematology and Oncology, University Medical Center\nHamburg-Eppendorf, Hamburg, Germany.\n(5)St Anna Kinderspital, Medical University Vienna, Vienna, Austria.\n(6)Research Center for Molecular Medicine (CeMM), Vienna, Austria.\n(7)UFR de Médecine Paris-Diderot, INSERM UMR_S1131-Institute Universitaire\nd'hématologie, Département de Génétique-UF de Génétique Moléculaire, Hôpital\nRobert Debré, Paris, France.\n\nChildren with P2RY8-CRLF2-positive acute lymphoblastic leukemia have an increased\nrelapse risk. Their mutational and transcriptional landscape, as well as the\nrespective patterns at relapse remain largely elusive. We, therefore, performed\nan integrated analysis of whole-exome and RNA sequencing in 41 major clone\nfusion-positive cases including 19 matched diagnosis/relapse pairs. We detected a\nvariety of frequently subclonal and highly instable JAK/STAT but also RTK/Ras\npathway-activating mutations in 76% of cases at diagnosis and virtually all\nrelapses. Unlike P2RY8-CRLF2 that was lost in 32% of relapses, all other genomic \nalterations affecting lymphoid development (58%) and cell cycle (39%) remained\nstable. Only IKZF1 alterations predominated in relapsing cases (P=0.001) and\nincreased from initially 36 to 58% in matched cases. IKZF1's critical role is\nfurther corroborated by its specific transcriptional signature comprising stem\ncell features with signs of impaired lymphoid differentiation, enhanced focal\nadhesion, activated hypoxia pathway, deregulated cell cycle and increased drug\nresistance. Our findings support the notion that P2RY8-CRLF2 is dispensable for\nrelapse development and instead highlight the prominent rank of IKZF1 for relapse\ndevelopment by mediating self-renewal and homing to the bone marrow niche.\nConsequently, reverting aberrant IKAROS signaling or its disparate programs\nemerges as an attractive potential treatment option in these leukemias.\n\nDOI: 10.1038/leu.2016.365 \nPMCID: PMC5508072\nPMID: 27899802  [Indexed for MEDLINE]",
     '21. Leuk Lymphoma. 2017 Apr;58(4):950-958. doi: 10.1080/10428194.2016.1219902. Epub\n2016 Nov 17.\n\nDiagnostic evaluation of RNA sequencing for the detection of genetic\nabnormalities associated with Ph-like acute lymphoblastic leukemia (ALL).\n\nYap KL(1), Furtado LV(2), Kiyotani K(1), Curran E(1), Stock W(1), McNeer JL(3),\nKadri S(2), Segal JP(2), Nakamura Y(1), Le Beau MM(1), Gurbuxani S(2), Raca G(1).\n\nAuthor information: \n(1)a Department of Medicine, Section of Hematology/Oncology , The University of\nChicago , Chicago , IL , USA.\n(2)b Department of Pathology , The University of Chicago , Chicago , IL , USA.\n(3)c Department of Pediatrics, Section of Pediatric Hematology/Oncology , The\nUniversity of Chicago , Chicago , IL , USA.\n\nPhiladelphia (Ph)-like acute lymphoblastic leukemia (ALL) is a molecular subtype \nof high-risk B-cell ALL characterized by formation of abnormal gene fusions\ninvolving tyrosine kinase (TK) and cytokine receptor genes and activation of TK\nsignaling. Because of the diversity of associated genetic changes, the detection \nof Ph-like ALL cases currently requires multiple cytogenetic and molecular\nassays; thus, our goal was to develop a consolidated workflow for detecting\ngenetic abnormalities in Ph-like ALL. We found that total and targeted RNA\nsequencing (RNAseq)-based approach allowed the detection of abnormal fusion\ntranscripts (EBF1-PDGFRB, P2RY8-CRLF2, RCSD1-ABL1, and RCSD1-ABL2). The\nbioinformatics algorithm accurately detected the fusion transcripts without prior\ninput about possible events. Additionally, we showed that RNAseq analysis enabled\nevaluation for disease-associated sequence variants in expressed transcripts.\nWhile total RNAseq can be a second tier approach allowing discovery of novel\ngenetic alterations, the targeted RNAseq workflow offers a clinically applicable \nmethod for the detection of fusion transcripts.\n\nDOI: 10.1080/10428194.2016.1219902 \nPMCID: PMC6077981\nPMID: 27855558  [Indexed for MEDLINE]',
     '22. Oncotarget. 2016 Dec 6;7(49):80664-80679. doi: 10.18632/oncotarget.13025.\n\nIntegrative analysis of DNA copy number, DNA methylation and gene expression in\nmultiple myeloma reveals alterations related to relapse.\n\nKrzeminski P(1)(2), Corchete LA(1), García JL(2), López-Corral L(1)(2), Fermiñán \nE(3), García EM(3), Martín AA(1), Hernández-Rivas JM(1)(2), García-Sanz R(1)(2), \nSan Miguel JF(4), Gutiérrez NC(1)(2).\n\nAuthor information: \n(1)Departamento de Hematología, Hospital Universitario, IBSAL, IBMCC (USAL-CSIC),\nSalamanca, Spain.\n(2)Centro de Investigación del Cáncer-IBMCC (USAL-CSIC), Salamanca, Spain.\n(3)Unidad de Genómica y Proteómica, Centro de Investigación del Cáncer-IBMCC\n(USAL-CSIC), Salamanca, Spain.\n(4)Clínica Universidad de Navarra, Centro de Investigaciones Médicas Aplicadas\n(CIMA), Pamplona, Spain.\n\nMultiple myeloma (MM) remains incurable despite the introduction of novel agents,\nand a relapsing course is observed in most patients. Although the development of \ngenomic technologies has greatly improved our understanding of MM pathogenesis,\nthe mechanisms underlying relapse have been less thoroughly investigated. In this\nstudy, an integrative analysis of DNA copy number, DNA methylation and gene\nexpression was conducted in matched diagnosis and relapse samples from MM\npatients. Overall, the acquisition of abnormalities at relapse was much more\nfrequent than the loss of lesions present at diagnosis, and DNA losses were\nsignificantly more frequent in relapse than in diagnosis samples. Interestingly, \ncopy number abnormalities involving more than 100 Mb of DNA at relapse\nsignificantly affect the gene expression of these samples, provoking a particular\nderegulation of the IL-8 pathway. On the other hand, no significant modifications\nof gene expression were observed in those samples with less than 100 Mb affected \nby chromosomal changes. Although several statistical approaches were used to\nidentify genes whose abnormal expression at relapse was regulated by methylation,\nonly two genes that were significantly deregulated in relapse samples (SORL1 and \nGLT1D1) showed a negative correlation between methylation and expression. Further\nanalysis revealed that DNA methylation was involved in regulating SORL1\nexpression in MM. Finally, relevant changes in gene expression observed in\nrelapse samples, such us downregulation of CD27 and P2RY8, were most likely not\npreceded by alterations in the corresponding DNA. Taken together, these results\nsuggest that the genomic heterogeneity described at diagnosis remains at relapse.\n\nDOI: 10.18632/oncotarget.13025 \nPMCID: PMC5348347\nPMID: 27811368  [Indexed for MEDLINE]',
     "23. Genes Chromosomes Cancer. 2017 Feb;56(2):135-146. doi: 10.1002/gcc.22421. Epub\n2016 Nov 5.\n\nPrognostic significance of P2RY8-CRLF2 and CRLF2 overexpression may vary across\nrisk subgroups of childhood B-cell acute lymphoblastic leukemia.\n\nDou H(1)(2), Chen X(3), Huang Y(4), Su Y(5), Lu L(6), Yu J(5), Yin Y(1)(2)(7),\nBao L(3)(8).\n\nAuthor information: \n(1)Department of Clinical Laboratory, Ministry of Education Key Laboratory of\nChild Development and Disorders, Children's Hospital of Chongqing Medical\nUniversity, Chongqing, China.\n(2)Key Laboratory of Pediatrics in Chongqing, Chongqing International Science and\nTechnology Cooperation Center for Child Development and Disorders, Children's\nHospital of Chongqing Medical University, Chongqing, China.\n(3)Center for Clinical Molecular Medicine, Ministry of Education Key Laboratory\nof Child Development and Disorders, Chongqing Key Laboratory of Pediatrics,\nChildren's Hospital of Chongqing Medical University, Chongqing, China.\n(4)Research Center for Immunity and Infectious Diseases, Ministry of Education\nKey Laboratory of Child Development and Disorders, Chongqing Key Laboratory of\nPediatrics, Children's Hospital of Chongqing Medical University, Chongqing,\nChina.\n(5)Department of Hematology and Oncology, Ministry of Education Key Laboratory of\nChild Development and Disorders, Chongqing Key Laboratory of Pediatrics,\nChildren's Hospital of Chongqing Medical University, Chongqing, China.\n(6)Department of Rheumatology, Huashan Hospital, Fudan University, Shanghai,\nChina.\n(7)Ministry of Education Key Laboratory of Clinical Laboratory Diagnostics,\nCollege of Laboratory Medicine, Chongqing Medical University, Chongqing, China.\n(8)Department of Pathology and Laboratory Medicine, Geisel School of Medicine\nDartmouth College, New Hampshire, USA.\n\nThe cytokine receptor-like factor 2 (CRLF2) gene plays an important role in early\nB-cell development. Aberrations in CRLF2 activate the JAK-STAT signaling pathway \nthat contributes to B-cell acute lymphoblastic leukemia (B-ALL). The prognostic\nsignificance of CRLF2 overexpression and P2RY8-CRLF2 fusion in various B-ALL risk\nsubgroups has not been well established. Two hundred seventy-one patients with\nnewly diagnosed childhood B-ALL were enrolled from a Chinese population. The\nprevalence of CRLF2 overexpression, CRLF2-P2RY8 fusion, CRLF2 F232C mutation, and\nJAK2 and IL7R mutational status were analyzed, and the prognostic impact of CRLF2\noverexpression and P2RY8-CRLF2 on B-ALL was evaluated by assessing their\ninfluence on overall survival and event-free survival. CRLF2 overexpression and\nP2RY8-CRLF2 were found in 19% and 10%, respectively, in the whole cohort. No\ncorrelation between CRLF2 overexpression and P2RY8-CRLF2 was observed. CRLF2\nF322C and IL7R mutations were not detected in B-ALL cases overexpressing CRLF2,\nand no JAK2 mutations were found in the whole cohort either. The results showed\nthat CRLF2 overexpression and P2RY8-CRLF2 were associated with a poor outcome in \nunselected B-ALL. Moreover, in an intermediate risk B-ALL subgroup P2RY8-CRLF2\nwas correlated with worse survival, whereas in high- and low-risk subgroups,\nCRLF2 overexpression predicted a poor outcome. Our findings suggest that\nP2RY8-CRLF2 is an independent prognostic indicator in intermediate risk B-ALL,\nwhile CRLF2 overexpression is correlated with an inferior outcome in high- or\nlow-risk B-ALL. Our study demonstrates that the impact of P2RY8-CRLF2 and CRLF2\noverexpression on B-ALL survival may differ across risk subgroups. © 2016 Wiley\nPeriodicals, Inc.\n\n© 2016 Wiley Periodicals, Inc.\n\nDOI: 10.1002/gcc.22421 \nPMID: 27637012  [Indexed for MEDLINE]",
     '24. J Mol Diagn. 2016 Jul;18(4):595-603. doi: 10.1016/j.jmoldx.2016.03.004. Epub 2016\nMay 6.\n\nGenome-Wide Single-Nucleotide Polymorphism Array Analysis Improves\nPrognostication of Acute Lymphoblastic Leukemia/Lymphoma.\n\nWang Y(1), Miller S(2), Roulston D(2), Bixby D(3), Shao L(4).\n\nAuthor information: \n(1)Clinical Cytogenetics Laboratory, University of Michigan, Ann Arbor, Michigan;\nDepartment of Pathology, Peking University First Hospital, Beijing, China.\n(2)Clinical Cytogenetics Laboratory, University of Michigan, Ann Arbor, Michigan.\n(3)Department of Pathology, and the Department of Internal Medicine, University\nof Michigan, Ann Arbor, Michigan.\n(4)Clinical Cytogenetics Laboratory, University of Michigan, Ann Arbor, Michigan.\nElectronic address: linashao@med.umich.edu.\n\nChromosomal abnormalities are important for the risk stratification of acute\nlymphoblastic leukemia/lymphoma (ALL). However, approximately 30% of pediatric\nand 50% of adult patients lack abnormalities with clinical relevance by\ntraditional cytogenetic analysis. We integrated cytogenetic, fluorescence in situ\nhybridization, and whole-genome single-nucleotide polymorphism array results from\n60 consecutive clinical ALL cases. By cytogenetic and/or fluorescence in situ\nhybridization analyses, recurring abnormalities with clinical relevance were\nobserved in 33 B-cell ALL (B-ALL), including t(9;22), hyperdiploidy, KMT2A\ntranslocation, ETV6-RUNX1, intrachromosomal amplification of chromosome 21, near \nhaploidy or low hypodiploidy, and t(8;22). Single-nucleotide polymorphism array\nanalysis found additional aberrations with prognostic or therapeutic implication \nin 21 B-ALL and two T-cell ALL, including IKZF1 deletion, intrachromosomal\namplification of chromosome 21 (one case with a normal karyotype), low\nhypodiploidy (two cases with a normal karyotype), and one case each with fusion\ngenes ETV6-NTRK3, CRLF2-P2RY8, NUP214-ABL1, and SET-NUP214. IKZF1 deletion was\nnoted in nine B-ALL with t(9;22), one B-ALL with t(4;11), five B-ALL with a\nnormal karyotype, and three B-ALL with nonrecurring karyotypic abnormalities.\nCombining single-nucleotide polymorphism array with chromosome and fluorescence\nin situ hybridization assays, the detection rate for clinically significant\nabnormal results increased from 56% to 75%. Whole-genome single-nucleotide\npolymorphism array analysis detects cytogenetically undetectable\nclinically\xa0significant aberrations and should be routinely applied at diagnosis\nof ALL.\n\nPublished by Elsevier Inc.\n\nDOI: 10.1016/j.jmoldx.2016.03.004 \nPMID: 27161658  [Indexed for MEDLINE]',
     '25. Leuk Res. 2016 Feb;41:36-42. doi: 10.1016/j.leukres.2015.11.018. Epub 2015 Dec\n23.\n\nCRLF2 overexpression identifies an unfavourable subgroup of adult B-cell\nprecursor acute lymphoblastic leukemia lacking recurrent genetic abnormalities.\n\nChiaretti S(1), Brugnoletti F(1), Messina M(1), Paoloni F(2), Fedullo AL(1),\nPiciocchi A(2), Elia L(1), Vitale A(1), Mauro E(3), Ferrara F(4), De Fabritiis\nP(5), Luppi M(6), Ronco F(7), De Propris MS(1), Raponi S(1), Kronnie GT(8),\nVignetti M(2), Guarini A(1), Foà R(9).\n\nAuthor information: \n(1)Hematology, Department of Cellular Biotechnologies and Hematology, "Sapienza" \nUniversity of Rome, Italy.\n(2)GIMEMA Data Center, Rome, Italy.\n(3)Division of Hematology, AOU Policlinico-OVE, University of Catania, Italy.\n(4)Division of Hematology and Stem Cell Transplantation Unit, Cardarelli\nHospital, Naples, Italy.\n(5)Hematology, Sant\'Eugenio Hospital, Rome, Italy.\n(6)Hematology Division, Department of Medical and Surgical Sciences, University\nof Modena and Reggio Emilia, Modena, Italy.\n(7)Hematology Unit, Azienda Ospedaliera Bianchi Melacrino Morelli, Reggio\nCalabria, Italy.\n(8)Laboratory of Oncohematology, Department of Women\'s and Children\'s Health,\nUniversity of Padova, Italy.\n(9)Hematology, Department of Cellular Biotechnologies and Hematology, "Sapienza" \nUniversity of Rome, Italy. Electronic address: rfoa@bce.uniroma1.it.\n\nBACKGROUND: A deregulated CRLF2 (d-CRLF2) expression was described in B-cell\nacute lymphoblastic leukemia without recurrent fusion genes (B-NEG ALL). While\nthe role of d-CRLF2 in children has been extensively described, little is known\nabout its role and impact in adult ALL.\nMETHODS: Expression levels of CRLF2 were evaluated by quantitative real-time PCR \nin 102 newly-diagnosed adult B-NEG ALL and correlated with the clinico-biological\ncharacteristics and outcome. Incidence and clinical impact of the P2RY8/CRLF2\ntranscript was also assessed.\nRESULTS: High CRLF2 levels, as continuous variable, were significantly associated\nwith hyperleucocytosis (p=0.0002) and thrombocytopenia (p=0.005); when a\ncut-point at ΔCt≤8 was applied, 35 cases (34.3%), mostly males (80%), proved\npositive for CRLF2 expression. High CRLF2 levels, as continuous or categorical\nvariable, were associated with a worse disease-free (p=0.003 and p=0.015) and\noverall survival (p=0.017 and 0.0038). Furthermore, when CRLF2 was analyzed as a \ncategorical variable, a high statistical association was found with IKZF1\ndeletion and mutations in the JAK/STAT pathway (p=0.001 and p<0.0001,\nrespectively). Finally, the P2RY8/CRLF2 transcript, identified in 8/102 patients \n(7.8%), was associated with a poor outcome.\nCONCLUSIONS: In adult B-NEG ALL, high CRLF2 expression is associated with\ndistinct clinico-biological features and an unfavourable prognosis in both\nunivariate and multivariate analysis; similarly, P2RY8/CRLF2 positivity\ncorrelates with a poor outcome. The quantification of CRLF2 is an important\nprognostic marker in adult B-lineage ALL without known genetic lesions.\n\nCopyright © 2015 Elsevier Ltd. All rights reserved.\n\nDOI: 10.1016/j.leukres.2015.11.018 \nPMID: 26754556  [Indexed for MEDLINE]',
     '26. J Exp Med. 2015 Dec 14;212(13):2213-22. doi: 10.1084/jem.20151250. Epub 2015 Nov \n16.\n\nThe G protein-coupled receptor P2RY8 and follicular dendritic cells promote\ngerminal center confinement of B cells, whereas S1PR3 can contribute to their\ndissemination.\n\nMuppidi JR(1), Lu E(2), Cyster JG(3).\n\nAuthor information: \n(1)Department of Microbiology and Immunology, University of California, San\nFrancisco, San Francisco, CA 94143 Department of Medicine, University of\nCalifornia, San Francisco, San Francisco, CA 94143 Howard Hughes Medical\nInstitute, University of California, San Francisco, San Francisco, CA 94143.\n(2)Department of Microbiology and Immunology, University of California, San\nFrancisco, San Francisco, CA 94143 Howard Hughes Medical Institute, University of\nCalifornia, San Francisco, San Francisco, CA 94143.\n(3)Department of Microbiology and Immunology, University of California, San\nFrancisco, San Francisco, CA 94143 Howard Hughes Medical Institute, University of\nCalifornia, San Francisco, San Francisco, CA 94143 Jason.Cyster@ucsf.edu.\n\nThe orphan Gα13-coupled receptor P2RY8 is mutated in human germinal center\n(GC)-derived lymphomas and was recently found to promote B cell association with \nGCs in a mouse model. Here we establish that P2RY8 promotes clustering of\nactivated B cells within follicles in a follicular dendritic cell (FDC)-dependent\nmanner. Although mice lack a P2RY8 orthologue, we show that mouse GC B cell\nclustering is also dependent on FDCs acting to support the function of a\nGα13-coupled receptor. Mutations in GNA13 and its downstream effector ARHGEF1 are\nassociated with the development of disseminated GC-derived lymphomas. We find\nthat egress of Gna13 mutant GC B cells from lymph nodes in the mouse depends on\nsphingosine-1-phosphate receptor-3. These findings provide evidence that FDCs\npromote GC confinement of both human and mouse GC B cells via Gα13-dependent\npathways, and they show that dissemination of Gα13-deficient GC B cells\nadditionally requires an egress-promoting receptor.\n\n© 2015 Muppidi et al.\n\nDOI: 10.1084/jem.20151250 \nPMCID: PMC4689170\nPMID: 26573295  [Indexed for MEDLINE]',
     "27. Br J Haematol. 2015 Sep;170(6):847-58. doi: 10.1111/bjh.13514. Epub 2015 May 27.\n\nThe clinical impact of IKZF1 deletions in paediatric B-cell precursor acute\nlymphoblastic leukaemia is independent of minimal residual disease stratification\nin Nordic Society for Paediatric Haematology and Oncology treatment protocols\nused between 1992 and 2013.\n\nOlsson L(1), Ivanov Öfverholm I(2), Norén-Nyström U(3), Zachariadis V(2),\nNordlund J(4), Sjögren H(5), Golovleva I(6), Nordgren A(2), Paulsson K(1), Heyman\nM(7), Barbany G(2), Johansson B(1)(8).\n\nAuthor information: \n(1)Division of Clinical Genetics, Department of Laboratory Medicine, Lund\nUniversity, Lund, Sweden.\n(2)Department of Molecular Medicine and Surgery and Centre for Molecular\nMedicine, Karolinska Institutet, Stockholm, Sweden.\n(3)Department of Clinical Sciences, Paediatrics, Umeå University, Umeå, Sweden.\n(4)Department of Medical Sciences, Molecular Medicine and Science for Life\nLaboratory, Uppsala University, Uppsala, Sweden.\n(5)Department of Clinical Chemistry and Transfusion Medicine, Institute of\nBiomedicine, The Sahlgrenska Academy, University of Gothenburg, Gothenburg,\nSweden.\n(6)Department of Medical Biosciences, Medical and Clinical Genetics, Umeå\nUniversity, Umeå, Sweden.\n(7)Department of Women's and Children's Health, Karolinska Institutet, Stockholm,\nSweden.\n(8)Department of Clinical Genetics, University and Regional Laboratories Region\nSkåne, Lund, Sweden.\n\nPaediatric B-cell precursor acute lymphoblastic leukaemias (BCP ALL) with IKZF1\ndeletions (∆IKZF1) are associated with a poor outcome. However, there are\nconflicting data as to whether ∆IKZF1 is an independent risk factor if minimal\nresidual disease (MRD) and other copy number alterations also are taken into\naccount. We investigated 334 paediatric BCP ALL, diagnosed 1992-2013 and treated \naccording to Nordic Society for Paediatric Haematology and Oncology ALL\nprotocols, with known IKZF1 status based on either single nucleotide polymorphism\narray (N\xa0=\xa0218) or multiplex ligation-dependent probe amplification (N\xa0=\xa0116)\nanalyses. ∆IKZF1, found in 15%, was associated with inferior 10-year\nprobabilities of event-free (60% vs. 83%; P\xa0<\xa00·001) and overall survival (pOS;\n73% vs. 89%; P\xa0=\xa00·001). Adjusting for known risk factors, including white blood \ncell (WBC) count and MRD, ∆IKZF1 was the strongest independent factor for relapse\nand death. ∆IKZF1 was present in 27% of cases with non-informative cytogenetics\n('BCP-other') and a poor 10-year pOS was particularly pronounced in this group\n(58% vs. 90%; P\xa0<\xa00·001). Importantly, neither MRD nor WBC count predicted events\nin the ∆IKZF1-positive cases. Co-occurrence of pseudoautosomal region 1 (PAR1)\ndeletions in Xp22.33/Yp11.32 (P2RY8-CRLF2) and ∆IKZF1 increased the risk of\nrelapse (75% vs. 30% for cases with only ∆IKZF1; P\xa0=\xa00·045), indicating that\nBCP-other ALL with both P2RY8-CRLF2 and ∆IKZF1 constitutes a particularly\nhigh-risk group.\n\n© 2015 John Wiley & Sons Ltd.\n\nDOI: 10.1111/bjh.13514 \nPMID: 26018335  [Indexed for MEDLINE]",
     '28. PLoS One. 2015 Apr 16;10(4):e0124246. doi: 10.1371/journal.pone.0124246.\neCollection 2015.\n\nLymphocyte gene expression signatures from patients and mouse models of\nhereditary hemochromatosis reveal a function of HFE as a negative regulator of\nCD8+ T-lymphocyte activation and differentiation in vivo.\n\nCosta M(1), Cruz E(2), Oliveira S(3), Benes V(4), Ivacevic T(4), Silva MJ(5),\nVieira I(3), Dias F(5), Fonseca S(5), Gonçalves M(5), Lima M(5), Leitão C(6),\nMuckenthaler MU(7), Pinto J(3), Porto G(8).\n\nAuthor information: \n(1)I3S - Instituto de Investigação e Inovação em Saúde, Universidade do Porto,\nPorto, Portugal; Basic and Clinical Research on Iron Biology, IBMC-Instituto de\nBiologia Molecular e Celular, Universidade do Porto, Porto, Portugal; Doctoral\nProgram in Biomedicine, Faculdade de Medicina, Universidade do Porto, Porto,\nPortugal.\n(2)I3S - Instituto de Investigação e Inovação em Saúde, Universidade do Porto,\nPorto, Portugal; Basic and Clinical Research on Iron Biology, IBMC-Instituto de\nBiologia Molecular e Celular, Universidade do Porto, Porto, Portugal; Clinical\nHematology, Santo António Hospital-Centro Hospitalar do Porto, Porto, Portugal.\n(3)I3S - Instituto de Investigação e Inovação em Saúde, Universidade do Porto,\nPorto, Portugal; Basic and Clinical Research on Iron Biology, IBMC-Instituto de\nBiologia Molecular e Celular, Universidade do Porto, Porto, Portugal.\n(4)Genomics Core Facility, European Molecular Biology Laboratory, Heidelberg,\nGermany.\n(5)Clinical Hematology, Santo António Hospital-Centro Hospitalar do Porto, Porto,\nPortugal.\n(6)Advanced Flow Cytometry Unit, IBMC-Instituto de Biologia Molecular e Celular, \nUniversidade do Porto, Porto, Portugal.\n(7)Departments of Pediatric Hematology, Oncology and Immunology, University of\nHeidelberg and Molecular Medicine Partnership Unit, Heidelberg, Germany.\n(8)I3S - Instituto de Investigação e Inovação em Saúde, Universidade do Porto,\nPorto, Portugal; Basic and Clinical Research on Iron Biology, IBMC-Instituto de\nBiologia Molecular e Celular, Universidade do Porto, Porto, Portugal; Clinical\nHematology, Santo António Hospital-Centro Hospitalar do Porto, Porto, Portugal;\nMolecular Immunology and Pathology, ICBAS-Instituto de Ciências Biomédicas Abel\nSalazar, Universidade do Porto, Porto, Portugal.\n\nAbnormally low CD8+ T-lymphocyte numbers is characteristic of some patients with \nhereditary hemochromatosis (HH), a MHC-linked disorder of iron overload. Both\nenvironmental and genetic components are known to influence CD8+ T-lymphocyte\nhomeostasis but the role of the HH associated protein HFE is still insufficiently\nunderstood. Genome-wide expression profiling was performed in peripheral blood\nCD8+ T lymphocytes from HH patients selected according to CD8+ T-lymphocyte\nnumbers and from Hfe-/- mice maintained either under normal or high iron diet\nconditions. In addition, T-lymphocyte apoptosis and cell cycle progression were\nanalyzed by flow cytometry in HH patients. HH patients with low CD8+ T-lymphocyte\nnumbers show a differential expression of genes related to lymphocyte\ndifferentiation and maturation namely CCR7, LEF1, ACTN1, NAA50, P2RY8 and FOSL2, \nwhose expression correlates with the relative proportions of naïve, central and\neffector memory subsets. In addition, expression levels of LEF1 and P2RY8 in\nmemory cells as well as the proportions of CD8+ T cells in G2/M cell cycle phase \nare significantly different in HH patients compared to controls. Hfe-/- mice do\nnot show alterations in CD8+ T-lymphocyte numbers but differential gene response \npatterns. We found an increased expression of S100a8 and S100a9 that is most\npronounced in high iron diet conditions. Similarly, CD8+ T lymphocytes from HH\npatients display higher S100a9 expression both at the mRNA and protein level.\nAltogether, our results support a role for HFE as a negative regulator of CD8+\nT-lymphocyte activation. While the activation markers S100a8 and S100a9 are\nstrongly increased in CD8+ T cells from both, Hfe-/- mice and HH patients, a\ndifferential profile of genes related to differentiation/maturation of CD8+ T\nmemory cells is evident in HH patients only. This supports the notion that HFE\ncontributes, at least in part, to the generation of low peripheral blood CD8+ T\nlymphocytes in HH.\n\nDOI: 10.1371/journal.pone.0124246 \nPMCID: PMC4399836\nPMID: 25880808  [Indexed for MEDLINE]',
     "29. Nature. 2014 Dec 11;516(7530):254-8. doi: 10.1038/nature13765. Epub 2014 Sep 28.\n\nLoss of signalling via Gα13 in germinal centre B-cell-derived lymphoma.\n\nMuppidi JR(1), Schmitz R(2), Green JA(3), Xiao W(2), Larsen AB(4), Braun SE(5),\nAn J(5), Xu Y(5), Rosenwald A(6), Ott G(7), Gascoyne RD(8), Rimsza LM(9), Campo\nE(10), Jaffe ES(11), Delabie J(12), Smeland EB(13), Braziel RM(14), Tubbs RR(15),\nCook JR(15), Weisenburger DD(16), Chan WC(17), Vaidehi N(4), Staudt LM(2), Cyster\nJG(5).\n\nAuthor information: \n(1)1] Department of Microbiology and Immunology, University of California, San\nFrancisco, California, 94143, USA [2] Department of Medicine, University of\nCalifornia, San Francisco, California 94143, USA [3] Howard Hughes Medical\nInstitute, University of California, San Francisco, California 94143, USA.\n(2)Lymphoid Malignancies Branch, Center for Cancer Research, National Cancer\nInstitute, National Institutes of Health, Bethesda, Maryland 20892, USA.\n(3)1] Department of Microbiology and Immunology, University of California, San\nFrancisco, California, 94143, USA [2] Howard Hughes Medical Institute, University\nof California, San Francisco, California 94143, USA [3] Immunology Program,\nMemorial Sloan-Kettering Cancer Center, New York, New York 10065, USA.\n(4)Division of Immunology, Beckman Research Institute of the City of Hope,\nDuarte, California 91010, USA.\n(5)1] Department of Microbiology and Immunology, University of California, San\nFrancisco, California, 94143, USA [2] Howard Hughes Medical Institute, University\nof California, San Francisco, California 94143, USA.\n(6)Department of Pathology, University of Würzburg, 97080 Würzburg, Germany.\n(7)1] Department of Clinical Pathology, Robert-Bosch-Krankenhaus, Auerbachstraße \n110, 70376 Stuttgart, Germany [2] Dr. Margarete Fischer-Bosch Institute for\nClinical Pharmacology, 70376 Stuttgart, Germany.\n(8)British Columbia Cancer Agency, Vancouver, British Columbia V5Z 1L3, Canada.\n(9)Department of Pathology, University of Arizona, Tucson, Arizona 85724, USA.\n(10)Hospital Clinic, University of Barcelona, 08036 Barcelona, Spain.\n(11)Laboratory of Pathology, Center for Cancer Research, National Cancer\nInstitute, National Institutes of Health, Bethesda, Maryland 20892, USA.\n(12)Pathology Clinic, Rikshospitalet University Hospital, 0372 Oslo, Norway.\n(13)1] Institute for Cancer Research, Rikshospitalet University Hospital,\nUniversity of Oslo, 0310 Oslo, Norway [2] Center for Cancer Biomedicine, Faculty \nDivision of the Norwegian Radium Hospital, University of Oslo, 0310 Oslo, Norway.\n(14)Oregon Health and Science University, Portland, Oregon 97239, USA.\n(15)Cleveland Clinic Pathology and Laboratory Medicine Institute, Cleveland, Ohio\n44195, USA.\n(16)Department of Pathology, City of Hope National Medical Center, Duarte,\nCalifornia 91010, USA.\n(17)1] Department of Pathology, University of Nebraska Medical Center, Omaha,\nNebraska 68198, USA [2] Department of Microbiology, University of Nebraska\nMedical Center, Omaha, Nebraska 68198, USA.\n\nGerminal centre B-cell-like diffuse large B-cell lymphoma (GCB-DLBCL) is a common\nmalignancy, yet the signalling pathways that are deregulated and the factors\nleading to its systemic dissemination are poorly defined. Work in mice showed\nthat sphingosine-1-phosphate receptor-2 (S1PR2), a Gα12 and Gα13 coupled\nreceptor, promotes growth regulation and local confinement of germinal centre B\ncells. Recent deep sequencing studies of GCB-DLBCL have revealed mutations in\nmany genes in this cancer, including in GNA13 (encoding Gα13) and S1PR2 (refs\n5,6, 7). Here we show, using in vitro and in vivo assays, that\nGCB-DLBCL-associated mutations occurring in S1PR2 frequently disrupt the\nreceptor's Akt and migration inhibitory functions. Gα13-deficient mouse germinal \ncentre B cells and human GCB-DLBCL cells were unable to suppress pAkt and\nmigration in response to S1P, and Gα13-deficient mice developed germinal centre\nB-cell-derived lymphoma. Germinal centre B cells, unlike most lymphocytes, are\ntightly confined in lymphoid organs and do not recirculate. Remarkably,\ndeficiency in Gα13, but not S1PR2, led to germinal centre B-cell dissemination\ninto lymph and blood. GCB-DLBCL cell lines frequently carried mutations in the\nGα13 effector ARHGEF1, and Arhgef1 deficiency also led to germinal centre B-cell \ndissemination. The incomplete phenocopy of Gα13- and S1PR2 deficiency led us to\ndiscover that P2RY8, an orphan receptor that is mutated in GCB-DLBCL and another \ngerminal centre B-cell-derived malignancy, Burkitt's lymphoma, also represses\ngerminal centre B-cell growth and promotes confinement via Gα13. These findings\nidentify a Gα13-dependent pathway that exerts dual actions in suppressing growth \nand blocking dissemination of germinal centre B cells that is frequently\ndisrupted in germinal centre B-cell-derived lymphoma.\n\nDOI: 10.1038/nature13765 \nPMCID: PMC4267955\nPMID: 25274307  [Indexed for MEDLINE]",
     '30. Nat Commun. 2014 Aug 8;5:4654. doi: 10.1038/ncomms5654.\n\nFrequent cases of RAS-mutated Down syndrome acute lymphoblastic leukaemia lack\nJAK2 mutations.\n\nNikolaev SI(1), Garieri M(2), Santoni F(3), Falconnet E(2), Ribaux P(2), Guipponi\nM(4), Murray A(5), Groet J(5), Giarin E(6), Basso G(6), Nizetic D(7), Antonarakis\nSE(8).\n\nAuthor information: \n(1)1] Department of Genetic Medicine and Development, University of Geneva\nMedical School, 1 rue Michel Servet, Geneva 1211, Switzerland [2].\n(2)Department of Genetic Medicine and Development, University of Geneva Medical\nSchool, 1 rue Michel Servet, Geneva 1211, Switzerland.\n(3)1] Department of Genetic Medicine and Development, University of Geneva\nMedical School, 1 rue Michel Servet, Geneva 1211, Switzerland [2] Geneva\nUniversity Hospitals-HUG, Service of Genetic Medicine, 4 Rue\nGabrielle-Perret-Gentil, Geneva 1211, Switzerland.\n(4)Geneva University Hospitals-HUG, Service of Genetic Medicine, 4 Rue\nGabrielle-Perret-Gentil, Geneva 1211, Switzerland.\n(5)1] The Blizard Institute, Barts and The London School of Medicine, Queen Mary \nUniversity of London, 4 Newark Street, London E1 2AT, UK [2] LonDownS Consortium,\nThe Wellcome Trust, London NW1 2BE, UK.\n(6)Dipartimento di Salute della Donna e del Bambino, University of Padua,\nIRP-Istituto di Ricerca Pediatrica-Fondazione Città della Speranza, Via N.\nGiustiniani 3, 35129 Padua, Italy.\n(7)1] The Blizard Institute, Barts and The London School of Medicine, Queen Mary \nUniversity of London, 4 Newark Street, London E1 2AT, UK [2] LonDownS Consortium,\nThe Wellcome Trust, London NW1 2BE, UK [3] Lee Kong Chian School of Medicine,\nNanyang Technological University, Unit 04-11, Proteos Building, 61 Biopolis\nDrive, Singapore 138673, Singapore [4].\n(8)1] Department of Genetic Medicine and Development, University of Geneva\nMedical School, 1 rue Michel Servet, Geneva 1211, Switzerland [2] IGE3 institute \nof Genetics and Genomics of Geneva, 1 rue Michel Servet, Geneva 1211, Switzerland\n[3].\n\nChildren with Down syndrome (DS) and acute lymphoblastic leukaemia (ALL) have\npoorer survival and more relapses than non-DS children with ALL, highlighting an \nurgent need for deeper mechanistic understanding of DS-ALL. Here, using\nfull-exome or cancer genes-targeted sequencing of 42 ALL samples from 39 DS\npatients, we uncover driver mutations in RAS, (KRAS and NRAS) recurring to a\nsimilar extent (15/42) as JAK2 (12/42) mutations or P2RY8-CRLF2 fusions (14/42). \nRAS mutations are almost completely mutually exclusive with JAK2 mutations\n(P=0.016), driving a combined total of two-thirds of analysed cases. Clonal\narchitecture analysis reveals that both RAS and JAK2 drove sub-clonal expansions \nprimarily initiated by CRLF2 rearrangements, and/or mutations in chromatin\nremodellers and lymphocyte differentiation factors. Remarkably, in 2/3 relapsed\ncases, there is a switch from a primary JAK2- or PTPN11-mutated sub-clone to a\nRAS-mutated sub-clone in relapse. These results provide important new insights\ninforming the patient stratification strategies for targeted therapeutic\napproaches for DS-ALL.\n\nDOI: 10.1038/ncomms5654 \nPMCID: PMC4665216\nPMID: 25105841  [Indexed for MEDLINE]',
     '31. Genes Chromosomes Cancer. 2014 Nov;53(11):902-10. doi: 10.1002/gcc.22201. Epub\n2014 Jul 16.\n\nGene alterations involving the CRLF2-JAK pathway and recurrent gene deletions in \nDown syndrome-associated acute lymphoblastic leukemia in Japan.\n\nHanada I(1), Terui K, Ikeda F, Toki T, Kanezaki R, Sato T, Kamio T, Kudo K,\nSasaki S, Takahashi Y, Hayashi Y, Inukai T, Kojima S, Koike K, Kosaka Y,\nKobayashi M, Imaizumi M, Mitsui T, Hori H, Hara J, Horibe K, Nagai J, Goto H, Ito\nE.\n\nAuthor information: \n(1)Department of Pediatrics, Hirosaki University Graduate School of Medicine,\nHirosaki, Japan.\n\nIn Western countries, gene alterations involving the CRLF2-JAK signaling pathway \nare identified in approximately 50-60% of patients with Down syndrome-associated \nacute lymphoblastic leukemia (DS-ALL), and this pathway is considered a potential\ntherapeutic target. The frequency of BTG1 deletions in DS-ALL is controversial.\nIKZF1 deletions, found in 20-30% of DS-ALL patients, are associated with a poor\noutcome and EBF1 deletions are very rare (∼2%). We analyzed 38 patients to\ndetermine the frequencies and clinical implications of CRLF2-JAK pathway genetic \nalterations and recurrent gene deletions in Japanese DS-ALL patients. We\nconfirmed a high incidence of P2RY8-CRLF2 (29%) and JAK2 mutations (16%), though \nthe frequency of P2RY8-CRLF2 was slightly lower than that in Western countries\n(∼50%). BTG1 deletions were common in our cohort (25%). IKZF1 deletions were\ndetected in 25% of patients and associated with shorter overall survival (OS).\nEBF1 deletions were found at an unexpectedly high frequency (16%), and at a\nsignificantly higher level in P2RY8-CRLF2-positive patients than in\nP2RY8-CRLF2-negative patients (44% vs. 4%, P=0.015). Deletions of CDKN2A/B and\nPAX5 were common in P2RY8-CRLF2-negative patients (48 and 39%, respectively) but \nnot in P2RY8-CRLF2-positive patients (11% each). Associations between these\ngenetic alterations and clinical characteristics were not observed except for\ninferior OS in patients with IKZF1 deletions. These results suggest that\ndifferences exist between the genetic profiles of DS-ALL patients in Japan and in\nWestern countries, and that P2RY8-CRLF2 and EBF1 deletions may cooperate in\nleukemogenesis in a subset of Japanese DS-ALL patients.\n\n© 2014 Wiley Periodicals, Inc.\n\nDOI: 10.1002/gcc.22201 \nPMID: 25044358  [Indexed for MEDLINE]',
     '32. Genes Chromosomes Cancer. 2014 Oct;53(10):815-23. doi: 10.1002/gcc.22190. Epub\n2014 Jun 17.\n\nAn overall characterization of pediatric acute lymphoblastic leukemia with CRLF2 \noverexpression.\n\nYano M(1), Imamura T, Asai D, Moriya-Saito A, Suenobu S, Hasegawa D, Deguchi T,\nHashii Y, Kawasaki H, Hori H, Kosaka Y, Kato K, Horibe K, Yumura-Yagi K, Hara J, \nMatsumoto K, Kiyokawa N, Oda M, Sato A; Japan Association of Childhood Leukemia\nStudy.\n\nAuthor information: \n(1)Department of Pediatrics, Kyoto Prefectural University of Medicine, Graduate\nSchool of Medical Science, Kyoto, Japan.\n\nFor an overall characterization of pediatric B-cell precursor acute lymphoblastic\nleukemia (BCPALL) with CRLF2 overexpression (OE), we conducted genetic analysis\nof CRLF2 in 167 pediatric BCPALL patients. CRLF2 OE was detected in 30 (18%) of\n167 patients, the P2RY8-CRLF2 fusion was identified in only 3 (1.8%) of 167\npatients, all of which demonstrated CRLF2 OE. Moreover, CRLF2 gain was identified\nin 18 (11%) of 167 patients. Messenger RNA sequencing revealed a novel fusion\ntranscript, CSF2RA-CRLF2, in a case with CRLF2 OE, suggesting that this fusion is\nassociated with CRLF2 OE. In survival analysis, no significant differences in\n5-year event-free survival (EFS) and overall survival were observed between\npatients with and without CRLF2 OE (70.7 vs. 75.4%, log rank P = 0.68 and 96.4\nvs. 82.1%, log rank P = 0.11, respectively). However, a significant difference in\n5-year EFS between CRLF2 OE patients with and without IKZF1 deletion was observed\n(44.4 vs. 83.1%, log rank P = 0.02). In multivariate analysis, only IKZF1\ndeletion was a significant predictor of inferior OS (hazard ratio: 2.427, P =\n0.04).These findings suggest that CRLF2 OE is not an independent prognostic\nfactor in pediatric BCPALL.\n\n© 2014 Wiley Periodicals, Inc.\n\nDOI: 10.1002/gcc.22190 \nPMID: 24935070  [Indexed for MEDLINE]',
     '33. Immunobiology. 2013 Nov;218(11):1345-53. doi: 10.1016/j.imbio.2013.07.001. Epub\n2013 Jul 15.\n\nAn mRNA atlas of G protein-coupled receptor expression during primary human\nmonocyte/macrophage differentiation and lipopolysaccharide-mediated activation\nidentifies targetable candidate regulators of inflammation.\n\nHohenhaus DM(1), Schaale K, Le Cao KA, Seow V, Iyer A, Fairlie DP, Sweet MJ.\n\nAuthor information: \n(1)Institute for Molecular Bioscience, The University of Queensland, Brisbane,\nQld 4072, Australia; Australian Infectious Disease Research Centre, The\nUniversity of Queensland, Brisbane, Qld 4072, Australia.\n\nG protein-coupled receptors (GPCRs) are among the most important targets in drug \ndiscovery. In this study, we used TaqMan Low Density Arrays to profile the full\nGPCR repertoire of primary human macrophages differentiated from monocytes using \neither colony stimulating factor-1 (CSF-1/M-CSF) (CSF-1 Mϕ) or granulocyte\nmacrophage colony stimulating factor (GM-CSF) (GM-CSF Mϕ). The overall trend was \na downregulation of GPCRs during monocyte to macrophage differentiation, but a\ncore set of 10 genes (e.g. LGR4, MRGPRF and GPR143) encoding seven transmembrane \nproteins were upregulated, irrespective of the differentiating agent used.\nSeveral of these upregulated GPCRs have not previously been studied in the\ncontext of macrophage biology and/or inflammation. As expected, CSF-1 Mϕ and\nGM-CSF Mϕ exhibited differential inflammatory cytokine profiles in response to\nthe Toll-like Receptor (TLR)4 agonist lipopolysaccharide (LPS). Moreover, 15\nGPCRs were differentially expressed between these cell populations in the basal\nstate. For example, EDG1 was expressed at elevated levels in CSF-1 Mϕ versus\nGM-CSF Mϕ, whereas the reverse was true for EDG6. 101 GPCRs showed differential\nregulation over an LPS time course, with 65 of these profiles being impacted by\nthe basal differentiation state (e.g. GPRC5A, GPRC5B). Only 14 LPS-regulated\nGPCRs showed asynchronous behavior (divergent LPS regulation) with respect to\ndifferentiation status. Thus, the differentiation state primarily affects the\nmagnitude of LPS-regulated expression, rather than causing major reprogramming of\nGPCR gene expression profiles. Several GPCRs showing differential profiles\nbetween CSF-1 Mϕ and GM-CSF Mϕ (e.g. P2RY8, GPR92, EMR3) have not been widely\ninvestigated in macrophage biology and inflammation. Strikingly, several closely \nrelated GPCRs displayed completely opposing patterns of regulation during\ndifferentiation and/or activation (e.g. EDG1 versus EDG6, LGR4 versus LGR7,\nGPRC5A versus GPRC5B). We propose that selective regulation of GPCR5A and GPCR5B \nin CSF-1 Mϕ contributes to skewing toward the M2 macrophage phenotype. Our\nanalysis of the GPCR repertoire expressed during primary human monocyte to\nmacrophage differentiation and TLR4-mediated activation provides a valuable new\nplatform for conducting future functional analyses of individual GPCRs in human\nmacrophage inflammatory pathways.\n\nCopyright © 2013 Elsevier GmbH. All rights reserved.\n\nDOI: 10.1016/j.imbio.2013.07.001 \nPMID: 23948647  [Indexed for MEDLINE]',
     '34. Cancer Med. 2013 Jun;2(3):412-9. doi: 10.1002/cam4.87. Epub 2013 May 9.\n\nIKZF1 deletion is associated with a poor outcome in pediatric B-cell precursor\nacute lymphoblastic leukemia in Japan.\n\nAsai D(1), Imamura T, Suenobu S, Saito A, Hasegawa D, Deguchi T, Hashii Y,\nMatsumoto K, Kawasaki H, Hori H, Iguchi A, Kosaka Y, Kato K, Horibe K,\nYumura-Yagi K, Hara J, Oda M; Japan Association of Childhood Leukemia Study\n(JACLS).\n\nAuthor information: \n(1)Department of Pediatrics, Kyoto Prefectural University of Medicine, Graduate\nSchool of Medical Science, Kyoto, Japan.\n\nGenetic alterations of Ikaros family zinc finger protein 1 (IKZF1), point\nmutations in Janus kinase 2 (JAK2), and overexpression of cytokine receptor-like \nfactor 2 (CRLF2) were recently reported to be associated with poor outcomes in\npediatric B-cell precursor (BCP)-ALL. Herein, we conducted genetic analyses of\nIKZF1 deletion, point mutation of JAK2 exon 16, 17, and 21, CRLF2 expression, the\npresence of P2RY8-CRLF2 fusion and F232C mutation in CRLF2 in 202 pediatric\nBCP-ALL patients newly diagnosed and registered in Japan Childhood Leukemia Study\nALL02 protocol to find out if alterations in these genes are determinants of poor\noutcome. All patients showed good response to initial prednisolone (PSL)\ntreatment. Ph⁺, infantile, and Down syndrome-associated ALL were excluded.\nDeletion of IKZF1 occurred in 19/202 patients (9.4%) and CRLF2 overexpression\noccurred in 16/107 (15.0%), which are similar to previous reports. Patients with \nIKZF1 deletion had reduced event-free survival (EFS) and overall survival (OS)\ncompared to those in patients without IKZF1 deletion (5-year EFS, 62.7% vs.\n88.8%, 5-year OS, 71.8% vs. 90.2%). Our data also showed significantly inferior\n5-year EFS (48.6% vs. 84.7%, log rank P = 0.0003) and 5-year OS (62.3% vs. 85.4%,\nlog rank P = 0.009) in NCI-HR patients (n = 97). JAK2 mutations and P2RY8-CRLF2\nfusion were rarely detected. IKZF1 deletion was identified as adverse prognostic \nfactor even in pediatric BCP-ALL in NCI-HR showing good response to PSL.\n\nDOI: 10.1002/cam4.87 \nPMCID: PMC3699852\nPMID: 23930217  [Indexed for MEDLINE]',
     '35. Haematologica. 2014 Jan;99(1):103-10. doi: 10.3324/haematol.2013.090225. Epub\n2013 Aug 2.\n\nHigh CD45 surface expression determines relapse risk in children with precursor\nB-cell and T-cell acute lymphoblastic leukemia treated according to the ALL-BFM\n2000 protocol.\n\nCario G(1), Rhein P, Mitlöhner R, Zimmermann M, Bandapalli OR, Romey R, Moericke \nA, Ludwig WD, Ratei R, Muckenthaler MU, Kulozik AE, Schrappe M, Stanulla M,\nKarawajew L.\n\nAuthor information: \n(1)g.cario@pediatrics.uni-kiel.de.\n\nFurther improvement of outcome in childhood acute lymphoblastic leukemia could be\nachieved by identifying additional high-risk patients who may benefit from\nintensified treatment. We earlier identified PTPRC (CD45) gene expression as a\npotential new stratification marker and now analyzed the prognostic relevance of \nCD45 protein expression. CD45 was measured by flow cytometry in 1065 patients\ntreated according to the ALL-BFM-2000 protocol. The 75(th) percentile was used as\ncut-off to distinguish a CD45-high from a CD45-low group. As mean CD45 expression\nwas significantly higher in T-cell acute lymphoblastic leukemia than in\nB-cell-precursor acute lymphoblastic leukemia (P<0.0001), the analysis was\nperformed separately in both groups. In B-cell-precursor acute lymphoblastic\nleukemia we observed a significant association of a high CD45 expression with\nolder age, high initial white blood cell count, ETV6/RUNX1 negativity, absence of\nhigh hyperdiploidy (P<0.0001), MLL/AF4 positivity (P=0.002), BCR/ABL1 positivity \n(P=0.007), prednisone poor response (P=0.002) and minimal residual disease\n(P<0.0001). In T-cell acute lymphoblastic leukemia we observed a significant\nassociation with initial white blood cell count (P=0.0003), prednisone poor\nresponse (P=0.01), and minimal residual disease (P=0.02). Compared to CD45-low\npatients, CD45-high patients had a lower event-free survival rate\n(B-cell-precursor acute lymphoblastic leukemia: 72 ± 3% versus 86 ± 1%, P<0.0001;\nT-cell acute lymphoblastic leukemia: 60 ± 8% versus 78 ± 4%, P=0.02), which was\nmainly attributable to a higher cumulative relapse incidence (B-cell-precursor\nacute lymphoblastic leukemia: 22 ± 3% versus 11 ± 1%, P<0.0001; T-cell acute\nlymphoblastic leukemia: 31 ± 8% versus 11 ± 3%, P=0.003) and kept its\nsignificance in multivariate analysis considering sex, age, initial white blood\ncell count, and minimal residual disease in B-cell-precursor- and T-cell acute\nlymphoblastic leukemia, and additionally presence of ETV6/RUNX1, MLL/AF4 and\nBCR/ABL1 rearrangements in B-cell-precursor acute lymphoblastic leukemia (P=0.002\nand P=0.025, respectively). Consideration of CD45 expression may serve as an\nadditional stratification tool in BFM-based protocols. (ClinicalTrials.gov\nidentifier: NCT00430118).\n\nDOI: 10.3324/haematol.2013.090225 \nPMCID: PMC4007934\nPMID: 23911702  [Indexed for MEDLINE]',
     "36. Blood. 2013 Jan 17;121(3):485-8. doi: 10.1182/blood-2012-04-422691. Epub 2012 Dec\n4.\n\nTyrosine kinome sequencing of pediatric acute lymphoblastic leukemia: a report\nfrom the Children's Oncology Group TARGET Project.\n\nLoh ML(1), Zhang J, Harvey RC, Roberts K, Payne-Turner D, Kang H, Wu G, Chen X,\nBecksfort J, Edmonson M, Buetow KH, Carroll WL, Chen IM, Wood B, Borowitz MJ,\nDevidas M, Gerhard DS, Bowman P, Larsen E, Winick N, Raetz E, Smith M, Downing\nJR, Willman CL, Mullighan CG, Hunger SP.\n\nAuthor information: \n(1)Department of Pediatrics and the Helen Diller Family Cancer Center, University\nof California-San Francisco, CA, USA.\n\nOne recently identified subtype of pediatric B-precursor acute lymphoblastic\nleukemia (ALL) has been termed BCR-ABL1-like or Ph-like because of similarity of \nthe gene expression profile to BCR-ABL1 positive ALL suggesting the presence of\nlesions activating tyrosine kinases, frequent alteration of IKZF1, and poor\noutcome. Prior studies demonstrated that approximately half of these patients had\ngenomic lesions leading to CRLF2 overexpression, with half of such cases\nharboring somatic mutations in the Janus kinases JAK1 and JAK2. To determine\nwhether mutations in other tyrosine kinases might also occur in ALL, we sequenced\nthe tyrosine kinome and downstream signaling genes in 45 high-risk pediatric ALL \ncases with either a Ph-like gene expression profile or other alterations\nsuggestive of activated kinase signaling. Aside from JAK mutations and 1 FLT3\nmutation, no somatic mutations were found in any other tyrosine kinases,\nsuggesting that alternative mechanisms are responsible for activated kinase\nsignaling in high-risk ALL.\n\nDOI: 10.1182/blood-2012-04-422691 \nPMCID: PMC3548168\nPMID: 23212523  [Indexed for MEDLINE]",
     "37. Br J Haematol. 2013 Feb;160(4):555-6. doi: 10.1111/bjh.12130. Epub 2012 Nov 12.\n\nNo prognostic impact of P2RY8-CRLF2 fusion in intermediate cytogenetic risk\nchildhood B-cell acute lymphoblastic leukaemia.\n\nKrawczyk J, Haslam K, Lynam P, Kelly J, Storey L, O'Marcaigh A, Langabeer SE,\nSmith OP.\n\nDOI: 10.1111/bjh.12130 \nPMID: 23145900  [Indexed for MEDLINE]",
     "38. Blood. 2012 Dec 20;120(26):5134-42. doi: 10.1182/blood-2012-07-443218. Epub 2012 \nOct 22.\n\nSmall sizes and indolent evolutionary dynamics challenge the potential role of\nP2RY8-CRLF2-harboring clones as main relapse-driving force in childhood ALL.\n\nMorak M(1), Attarbaschi A, Fischer S, Nassimbeni C, Grausenburger R, Bastelberger\nS, Krentz S, Cario G, Kasper D, Schmitt K, Russell LJ, Pötschger U, Stanulla M,\nEckert C, Mann G, Haas OA, Panzer-Grümayer R.\n\nAuthor information: \n(1)Children's Cancer Research Institute, St Anna Kinderkrebsforschung, Vienna,\nAustria.\n\nErratum in\n    Blood. 2013 Aug 15;122(7):1328.\n\nThe P2RY8-CRLF2 fusion defines a particular relapse-prone subset of childhood\nacute lymphoblastic leukemia (ALL) in Italian Association of Pediatric Hematology\nand Oncology Berlin-Frankfurt-Münster (AIEOP-BFM) 2000 protocols. To investigate \nwhether and to what extent different clone sizes influence disease and relapse\ndevelopment, we quantified the genomic P2RY8-CRLF2 fusion product and correlated \nit with the corresponding CRLF2 expression levels in patients enrolled in the\nBFM-ALL 2000 protocol in Austria. Of 268 cases without recurrent chromosomal\ntranslocations and high hyperdiploidy, representing approximately 50% of all\ncases, 67 (25%) were P2RY8-CRLF2 positive. The respective clone sizes were ≥ 20% \nin 27% and < 20% in 73% of them. The cumulative incidence of relapse of the\nentire fusion-positive group was clone size independent and significantly higher \nthan that of the fusion-negative group (35% ± 8% vs 13% ± 3%, P = .008) and\nprimarily confined to the non-high-risk group. Of 22 P2RY8-CRLF2-positive\ndiagnosis/relapse pairs, only 4/8 had the fusion-positive dominant clone\nconserved at relapse, whereas none of the original 14 fusion-positive small\nclones reappeared as the dominant relapse clone. We conclude that the majority of\nP2RY8-CRLF2-positive clones are small at diagnosis and virtually never generate a\ndominant relapse clone. Our findings therefore suggest that P2RY8-CRLF2-positive \nclones do not have the necessary proliferative or selective advantage to evolve\ninto a disease-relevant relapse clone.\n\nDOI: 10.1182/blood-2012-07-443218 \nPMCID: PMC4194314\nPMID: 23091296  [Indexed for MEDLINE]",
     '39. Haematologica. 2013 Mar;98(3):428-32. doi: 10.3324/haematol.2011.056135. Epub\n2012 Aug 8.\n\nIKZF1 deletion is an independent predictor of outcome in pediatric acute\nlymphoblastic leukemia treated according to the ALL-BFM 2000 protocol.\n\nDörge P(1), Meissner B, Zimmermann M, Möricke A, Schrauder A, Bouquin JP, Schewe \nD, Harbott J, Teigler-Schlegel A, Ratei R, Ludwig WD, Koehler R, Bartram CR,\nSchrappe M, Stanulla M, Cario G.\n\nAuthor information: \n(1)Department of Pediatrics, University Medical Centre Schleswig-Holstein, Kiel, \nGermany.\n\nIKZF1 gene deletions have been associated with a poor outcome in pediatric\nprecursor B-cell acute lymphoblastic leukemia. To assess the prognostic relevance\nof IKZF1 deletions for patients treated on Berlin-Frankfurt-Münster Study Group\ntrial ALL-BFM 2000, we screened 694 diagnostic acute lymphoblastic leukemia\nsamples by Multiplex Ligation-dependent Probe Amplification. Patients whose\nleukemic cells bore IKZF1 deletions had a lower 5-year event-free survival\n(0.69±0.05 vs. 0.85±0.01; P<0.0001) compared to those without, mainly due to a\nhigher cumulative incidence of relapses (0.21±0.04 vs. 0.10±0.01; P=0.001).\nAlthough IKZF1 deletions were significantly associated with the P2RY8-CRLF2\nrearrangement, their prognostic value was found to be independent from this\nassociation. Thus, IKZF1 deletion is an independent predictor of treatment\noutcome and a strong candidate marker for integration in future treatment\nstratification strategies on ALL-BFM protocols. Clinicaltrials.gov identifier:\nNCT00430118.\n\nDOI: 10.3324/haematol.2011.056135 \nPMCID: PMC3659952\nPMID: 22875627  [Indexed for MEDLINE]',
     '40. J Clin Oncol. 2012 Sep 1;30(25):3100-8. doi: 10.1200/JCO.2011.40.3907. Epub 2012 \nJul 30.\n\nIGH@ translocations, CRLF2 deregulation, and microdeletions in adolescents and\nadults with acute lymphoblastic leukemia.\n\nMoorman AV(1), Schwab C, Ensor HM, Russell LJ, Morrison H, Jones L, Masic D,\nPatel B, Rowe JM, Tallman M, Goldstone AH, Fielding AK, Harrison CJ.\n\nAuthor information: \n(1)Leukaemia Research Cytogenetics Group, Northern Institute for Cancer Research,\nLevel 5, Sir James Spence Institute, Royal Victoria Infirmary, Queen Victoria Rd,\nNewcastle-upon-Tyne, NE1 4LP, United Kingdom. anthony.moorman@ncl.ac.uk\n\nPURPOSE: To determine the prevalence and prognostic impact of significant acute\nlymphoblastic leukemia (ALL) -related genes: CRLF2 deregulation (CRLF2-d), IGH@\ntranslocations (IGH@-t), and deletions of CDKN2A/B, IKZF1, PAX5, ETV6, RB1, BTG1,\nand EBF1 in adolescents and adults.\nPATIENTS AND METHODS: The cohort comprised 454 patients (age 15 to 60 years old) \ntreated on the multicenter United Kingdom Acute Lymphoblastic Leukaemia Trial\nXII/Eastern Cooperative Oncology Group 2993 trial (UKALLXII/ECOG2993) with\nPhiladelphia-negative B-cell precursor ALL. Fluorescent in situ hybridization and\nmultiplex ligation-dependent probe amplification were used to detect these\ngenetic alterations.\nRESULTS: Twenty patients (5%) had CRLF2-d (P2RY8-CRLF2, n = 7; IGH@-CRLF2, n =\n13), and 36 patients (8%) harbored an IGH@-t with a different partner gene. There\nwas little overlap between IGH@-t, CRLF2-d, and established chromosomal\nabnormalities. Deletions of CDKN2A/B, IKZF1, PAX5, ETV6, RB1, BTG1, or EBF1 were \nprevalent with 101 (33%) of 304 patients harboring one and 102 (33%) harboring\ntwo or more alterations, occurring with varying frequency in all cytogenetic\nsubgroups. The 5-year event-free survival, relapse-free survival (RFS), and\noverall survival (OS) rates for the whole cohort were 40%, 55%, and 43%,\nrespectively. Patients with CRLF2-d, IGH@-t, and IKZF1 deletions were associated \nwith an inferior outcome in univariate but not multivariate analysis. In\nparticular, CRLF2-d patients had a lower RFS compared with other patients (30%), \nwhereas those with IGH@-t or IKZF1 deletions had a lower OS (27% and 35%,\nrespectively).\nCONCLUSION: CRLF2-d and IGH@-t represent distinct subtypes of adolescent and\nadult ALL. Deletions of key B-cell differentiation and cell cycle control genes\nare highly prevalent but vary in frequency by cytogenetic subgroup. CRLF2-d,\nIGH@-t, and IKZF1 deletions are associated with poor outcome in adolescent and\nadult ALL.\n\nDOI: 10.1200/JCO.2011.40.3907 \nPMID: 22851563  [Indexed for MEDLINE]',
     "41. Br J Haematol. 2012 Sep;158(6):772-7. doi: 10.1111/j.1365-2141.2012.09221.x. Epub\n2012 Jul 23.\n\nTreatment outcome of CRLF2-rearranged childhood acute lymphoblastic leukaemia: a \ncomparative analysis of the AIEOP-BFM and UK NCRI-CCLG study groups.\n\nAttarbaschi A(1), Morak M, Cario G, Cazzaniga G, Ensor HM, te Kronnie T, Bradtke \nJ, Mann G, Vendramini E, Palmi C, Schwab C, Russell LJ, Schrappe M, Conter V,\nMitchell CD, Strehl S, Zimmermann M, Pötschger U, Harrison CJ, Stanulla M,\nPanzer-Grümayer R, Haas OA, Moorman AV; Associazione Italiana di Ematologia ed\nOncologia Pediatrica (AIEOP)-Berlin-Frankfurt-Münster (BFM) Study Group and\nNational Cancer Research Institute (NCRI)-Children's Cancer and Leukaemia (CCLG) \nStudy Group.\n\nAuthor information: \n(1)Paediatric Haematology and Oncology, Department of Paediatrics, St. Anna\nChildren's Hospital, Medical University of Vienna, Vienna, Austria.\nandishe.attarbaschi@stanna.at\n\nThe prognostic relevance of CRLF2 -rearrangements in childhood acute B-cell\nprecursor lymphoblastic leukaemia (ALL), was assessed by a comparative analysis\nof 114 non-Down-syndrome patients (99 P2RY8-CRLF2+ , 15 IGH@-CRLF2+ ), 76 from\nthe AIEOP-BFM ALL 2000 and 38 from the MRC ALL97 trials. The 6-year cumulative\nrelapse incidence of P2RY8-CRLF2+ patients treated on the two trials was not\nstatistically different: 0·37 ± 0·06 vs. 0·25 ± 0·08 (P = 0·194). In contrast,\n0/9 IGH@-CRLF2+ AIEOP-BFM, but 5/6 ALL97 patients relapsed. Conclusively,\nP2RY8-CRLF2+ patients had an intermediate protocol-independent outcome while the \ndifferent prognosis of IGH@-CRLF2+ patients could be related to the different\nstructures of the applied treatment protocols.\n\n© 2012 Blackwell Publishing Ltd.\n\nDOI: 10.1111/j.1365-2141.2012.09221.x \nPMID: 22816614  [Indexed for MEDLINE]",
     '42. Leukemia. 2012 Oct;26(10):2245-53. doi: 10.1038/leu.2012.101. Epub 2012 Apr 9.\n\nPoor prognosis for P2RY8-CRLF2 fusion but not for CRLF2 over-expression in\nchildren with intermediate risk B-cell precursor acute lymphoblastic leukemia.\n\nPalmi C(1), Vendramini E, Silvestri D, Longinotti G, Frison D, Cario G, Shochat\nC, Stanulla M, Rossi V, Di Meglio AM, Villa T, Giarin E, Fazio G, Leszl A,\nSchrappe M, Basso G, Biondi A, Izraeli S, Conter V, Valsecchi MG, Cazzaniga G, Te\nKronnie G.\n\nAuthor information: \n(1)Centro Ricerca Tettamanti, Clinica Pediatrica, Università di Milano Bicocca,\nOspedale San Gerardo, Monza, Italy.\n\nPediatric B-cell precursor acute lymphoblastic leukemia (BCP-ALL) has achieved an\n80% cure rate as a result of a risk-adapted therapy largely based on minimal\nresidual disease (MRD) monitoring. However, relapse is still the most frequent\nadverse event, occurring mainly in the patients with intermediate MRD levels\n(intermediate risk, IR), emphasizing the need for new prognostic markers. We\nanalyzed the prognostic impact of cytokine receptor-like factor 2 (CRLF2)\nover-expression and P2RY8-CRLF2 fusion in 464 BCP-ALL patients (not affected by\nDown syndrome and BCR-ABL negative) enrolled in the AIEOP-BFM ALL2000 study in\nItaly. In 22/464 (4.7%) samples, RQ-PCR showed CRLF2 over-expression (≥20 times\nhigher than the overall median). P2RY8-CRLF2 fusion was detected in 22/365 (6%)\ncases, with 10/22 cases also showing CRLF2 over-expression. P2RY8-CRLF2 fusion\nwas the most relevant prognostic factor independent of CRLF2 over-expression with\na threefold increase in risk of relapse. Significantly, the cumulative incidence \nof relapse of the P2RY8-CRLF2 + patients in the IR group was high (61.1% ± 12.9\nvs 17.6% ± 2.6, P<0.0001), similar to high-risk patients in AIEOP-BFM ALL2000\nstudy. These results were confirmed in a cohort of patients treated in Germany.\nIn conclusion, P2RY8-CRLF2 identifies a subset of BCP-ALL patients currently\nstratified as IR that could be considered for treatment intensification.\n\nDOI: 10.1038/leu.2012.101 \nPMID: 22484421  [Indexed for MEDLINE]',
     "43. Blood. 2012 Apr 12;119(15):3512-22. doi: 10.1182/blood-2011-11-394221. Epub 2012 \nFeb 24.\n\nOutcome modeling with CRLF2, IKZF1, JAK, and minimal residual disease in\npediatric acute lymphoblastic leukemia: a Children's Oncology Group study.\n\nChen IM(1), Harvey RC, Mullighan CG, Gastier-Foster J, Wharton W, Kang H,\nBorowitz MJ, Camitta BM, Carroll AJ, Devidas M, Pullen DJ, Payne-Turner D, Tasian\nSK, Reshmi S, Cottrell CE, Reaman GH, Bowman WP, Carroll WL, Loh ML, Winick NJ,\nHunger SP, Willman CL.\n\nAuthor information: \n(1)Cancer Center and Department of Internal Medicine, University of New Mexico,\nAlbuquerque, NM 87131-001, USA.\n\nAs controversy exists regarding the prognostic significance of genomic\nrearrangements of CRLF2 in pediatric B-precursor acute lymphoblastic leukemia\n(ALL) classified as standard/intermediate-risk (SR) or high-risk (HR), we\nassessed the prognostic significance of CRLF2 mRNA expression, CRLF2 genomic\nlesions (IGH@-CRLF2, P2RY8-CRLF2, CRLF2 F232C), deletion/mutation in genes\nfrequently associated with high CRLF2 expression (IKZF1, JAK, IL7R), and minimal \nresidual disease (MRD) in 1061 pediatric ALL patients (499 HR and 562 SR) on COG \nTrials P9905/P9906. Whereas very high CRLF2 expression was found in 17.5% of\ncases, only 51.4% of high CRLF2 expressors had CRLF2 genomic lesions. The\nmechanism underlying elevated CRLF2 expression in cases lacking known genomic\nlesions remains to be determined. All CRLF2 genomic lesions and virtually all JAK\nmutations were found in high CRLF2 expressors, whereas IKZF1 deletions/mutations \nwere distributed across the full cohort. In multivariate analyses, NCI risk\ngroup, MRD, high CRLF2 expression, and IKZF1 lesions were associated with\nrelapse-free survival. Within HR ALL, only MRD and CRLF2 expression predicted a\npoorer relapse-free survival; no difference was seen between cases with or\nwithout CRLF2 genomic lesions. Thus, high CRLF2 expression is associated with a\nvery poor outcome in high-risk, but not standard-risk, ALL. This study is\nregistered at www.clinicaltrials.gov as NCT00005596 and NCT00005603.\n\nDOI: 10.1182/blood-2011-11-394221 \nPMCID: PMC3325039\nPMID: 22368272  [Indexed for MEDLINE]",
     '44. Proc Natl Acad Sci U S A. 2012 Mar 6;109(10):3879-84. doi:\n10.1073/pnas.1121343109. Epub 2012 Feb 17.\n\nDiscovery and prioritization of somatic mutations in diffuse large B-cell\nlymphoma (DLBCL) by whole-exome sequencing.\n\nLohr JG(1), Stojanov P, Lawrence MS, Auclair D, Chapuy B, Sougnez C,\nCruz-Gordillo P, Knoechel B, Asmann YW, Slager SL, Novak AJ, Dogan A, Ansell SM, \nLink BK, Zou L, Gould J, Saksena G, Stransky N, Rangel-Escareño C,\nFernandez-Lopez JC, Hidalgo-Miranda A, Melendez-Zajgla J, Hernández-Lemus E,\nSchwarz-Cruz y Celis A, Imaz-Rosshandler I, Ojesina AI, Jung J, Pedamallu CS,\nLander ES, Habermann TM, Cerhan JR, Shipp MA, Getz G, Golub TR.\n\nAuthor information: \n(1)Eli and Edythe Broad Institute, Cambridge, MA 02412, USA.\n\nTo gain insight into the genomic basis of diffuse large B-cell lymphoma (DLBCL), \nwe performed massively parallel whole-exome sequencing of 55 primary tumor\nsamples from patients with DLBCL and matched normal tissue. We identified\nrecurrent mutations in genes that are well known to be functionally relevant in\nDLBCL, including MYD88, CARD11, EZH2, and CREBBP. We also identified somatic\nmutations in genes for which a functional role in DLBCL has not been previously\nsuspected. These genes include MEF2B, MLL2, BTG1, GNA13, ACTB, P2RY8, PCLO, and\nTNFRSF14. Further, we show that BCL2 mutations commonly occur in patients with\nBCL2/IgH rearrangements as a result of somatic hypermutation normally occurring\nat the IgH locus. The BCL2 point mutations are primarily synonymous, and likely\ncaused by activation-induced cytidine deaminase-mediated somatic hypermutation,\nas shown by comprehensive analysis of enrichment of mutations in WRCY target\nmotifs. Those nonsynonymous mutations that are observed tend to be found outside \nof the functionally important BH domains of the protein, suggesting that strong\nnegative selection against BCL2 loss-of-function mutations is at play. Last, by\nusing an algorithm designed to identify likely functionally relevant but\ninfrequent mutations, we identify KRAS, BRAF, and NOTCH1 as likely drivers of\nDLBCL pathogenesis in some patients. Our data provide an unbiased view of the\nlandscape of mutations in DLBCL, and this in turn may point toward new\ntherapeutic strategies for the disease.\n\nDOI: 10.1073/pnas.1121343109 \nPMCID: PMC3309757\nPMID: 22343534  [Indexed for MEDLINE]',
     '45. Genes Chromosomes Cancer. 2012 Feb;51(2):196-206. doi: 10.1002/gcc.20944. Epub\n2011 Nov 10.\n\nHigh frequency of BTG1 deletions in acute lymphoblastic leukemia in children with\ndown syndrome.\n\nLundin C(1), Hjorth L, Behrendtz M, Nordgren A, Palmqvist L, Andersen MK,\nBiloglav A, Forestier E, Paulsson K, Johansson B.\n\nAuthor information: \n(1)Department of Clinical Genetics, University and Regional Laboratories, Skåne\nUniversity Hospital, Lund University, Sweden. catarina.lundin@med.lu.se\n\nPrevious cytogenetic studies of myeloid and acute lymphoblastic leukemias in\nchildren with Down syndrome (ML-DS and DS-ALL) have revealed significant\ndifferences in abnormality patterns between such cases and acute leukemias in\ngeneral. Also, certain molecular genetic aberrations characterize DS-related\nleukemias, such as GATA1 mutations in ML-DS and deregulation of the CRLF2 gene in\nDS-ALL. Whether microdeletions/microduplications also vary between DS and non-DS \ncases is presently unclear. To address this issue, we performed single nucleotide\npolymorphism array analyses of eight pediatric ML-DS and 17 B-cell precursor\nDS-ALL. In the ML-DS cases, a total of 29 imbalances (20 gains and nine losses)\nand two partial uniparental isodisomies (pUPDs) were detected. None of the 11\nsmall (defined as <10 Mb) imbalances were recurrent, nor were the pUPDs, whereas \nof the 18 large aberrations, three were recurrent-dup(1q), +8 and +21. In\ncontrast, several frequent changes were identified in the DS-ALL cases, which\nharbored 82 imbalances (30 gains and 52 losses) and four pUPDs. Of the 40 large\nchanges, 28 were gains and 12 losses, with +X, dup(Xq), dup(1q), del(7p),\ndup(8q), del(9p), dup(9p), del(12p), dup(17q), and +21 being recurrent. Of the 40\nmicrodeletions identified, several targeted specific genes, with the following\nbeing repeatedly deleted: BTG1 and CDKN2A/B (29% of cases), ETV6, IKZF1, PAX5 and\nSERP2 (18%), and BTLA, INPP4B, P2RY8, and RB1 (12%). Loss of the SERP2 and INPP4B\ngenes, encoding the stress-associated endoplasmic reticulum protein family member\n2 and the inositol polyphosphate 4-phosphatase-II, respectively, has previously\nnever been implicated in leukemia. Although deletions of the other genes have\nbeen associated with ALL, the high frequency of BTG1 loss is a novel finding.\nSuch deletions may characterize a clinical subgroup of DS-ALL, comprising mainly \nboys with a high median age. In conclusion, ML-DS and DS-ALL are genetically\ndistinct, with mainly gains in ML-DS and deletions in DS-ALL. Furthermore, DS-ALL\nis characterized by several recurrent gene deletions, with BTG1 loss being\nparticularly frequent.\n\nCopyright © 2011 Wiley-Liss, Inc.\n\nDOI: 10.1002/gcc.20944 \nPMID: 22072402  [Indexed for MEDLINE]',
     '46. Blood. 2011 Jun 23;117(25):6848-55. doi: 10.1182/blood-2011-01-329961. Epub 2011 \nApr 28.\n\nGenomic characterization implicates iAMP21 as a likely primary genetic event in\nchildhood B-cell precursor acute lymphoblastic leukemia.\n\nRand V(1), Parker H, Russell LJ, Schwab C, Ensor H, Irving J, Jones L, Masic D,\nMinto L, Morrison H, Ryan S, Robinson H, Sinclair P, Moorman AV, Strefford JC,\nHarrison CJ.\n\nAuthor information: \n(1)Northern Institute for Cancer Research, Newcastle University, Newcastle upon\nTyne, UK.\n\nIntrachromosomal amplification of chromosome 21 (iAMP21) defines a distinct\nsubgroup of childhood B-cell precursor acute lymphoblastic leukemia (BCP-ALL)\nthat has a dismal outcome when treated with standard therapy. For improved\ndiagnosis and risk stratification, the initiating genetic events need to be\nelucidated. To investigate the genetic basis of BCP-ALL, genomes of 94 iAMP21\npatients were interrogated by arrays, FISH, and multiplex ligation-dependent\nprobe amplification. Most copy number alterations targeted chromosome 21,\nreinforcing the complexity of this chromosome. The common region of amplification\non chromosome 21 was refined to a 5.1-mb region that included RUNX1, miR-802, and\ngenes mapping to the Down syndrome critical region. Recurrent abnormalities\naffecting genes in key pathways were identified: IKZF1 (22%), CDKN2A/B (17%),\nPAX5 (8%), ETV6 (19%), and RB1 (37%). Investigation of clonal architecture\nprovided evidence that these abnormalities, and P2RY8-CRLF2, were secondary to\nchromosome 21 rearrangements. Patient outcome was uniformly poor with standard\ntherapy irrespective of the presence or absence of these changes. This study has \nprovided evidence that chromosome 21 instability is the only anomaly among those \nso far investigated that is common to all iAMP21 patients, and therefore the\ninitiating event is likely to be found among the complex structural\nrearrangements of this abnormal chromosome.\n\nDOI: 10.1182/blood-2011-01-329961 \nPMID: 21527530  [Indexed for MEDLINE]',
     '47. Blood. 2011 Feb 17;117(7):2129-36. doi: 10.1182/blood-2010-07-297135. Epub 2010\nNov 24.\n\nDemographic, clinical, and outcome features of children with acute lymphoblastic \nleukemia and CRLF2 deregulation: results from the MRC ALL97 clinical trial.\n\nEnsor HM(1), Schwab C, Russell LJ, Richards SM, Morrison H, Masic D, Jones L,\nKinsey SE, Vora AJ, Mitchell CD, Harrison CJ, Moorman AV.\n\nAuthor information: \n(1)Leukaemia Research Cytogenetics Group, Northern Institute for Cancer Research,\nNewcastle University, Newcastle-upon-Tyne, UK.\n\nErratum in\n    Blood. 2011 May 5;117(18):5009.\n\nComment in\n    Blood. 2011 Feb 17;117(7):2081-2.\n\nDeregulated expression of CRLF2 (CRLF2-d) arises via its juxtaposition to the\nIGH@ enhancer or P2RY8 promoter. Among 865 BCP-ALL children treated on MRC ALL97,\n52 (6%) had CRLF2-d, but it was more prevalent among Down syndrome patients\n(54%). P2RY8-CRLF2 (n = 43) was more frequent than IGH@-CRLF2 (n = 9). CRLF2-d\nwas not associated with age, sex, or white cell count, but IGH@-CRLF2 patients\nwere older than P2RY8-CRLF2 patients (median 8 vs 4 years, P = .0017). Patients\nwith CRLF2-d were more likely to present with enlarged livers and spleens (38% vs\n18%, P < .001). CRLF2-d was not seen in conjunction with established chromosomal \ntranslocations but 6 (12%) cases had high hyperdiploidy, and 5 (10%) had iAMP21. \nUnivariate analysis suggested that CRLF2-d was associated with an inferior\noutcome: (event-free survival [EFS] hazard ratio 2.27 [95% confidence interval\n1.48-3.47], P < .001; OS 3.69 [2.34-5.84], P < .001). However, multivariate\nanalysis indicated that its effect was mediated by other risk factors such as\ncytogenetics and DS status (EFS 1.45 [0.88-2.39], P = .140; OS 1.90 [1.08-3.36], \nP = .027). Although the outcome of IGH@-CRLF2 patients appeared inferior compared\nwith P2RY8-CRLF2 patients, the result was not significant (EFS 2.69 [1.15-6.31], \nP = .023; OS 2.86 [1.15-6.79], P = .021). Therefore, we concluded that patients\nwith CRLF2-d should be classified into the intermediate cytogenetic risk group.\n\nDOI: 10.1182/blood-2010-07-297135 \nPMID: 21106984  [Indexed for MEDLINE]',
     '48. Blood. 2010 Sep 16;116(11):1993-4. doi: 10.1182/blood-2010-05-286492.\n\nt(X;14)(p22;q32)/t(Y;14)(p11;q32) CRLF2-IGH translocations from human B-lineage\nALLs involve CpG-type breaks at CRLF2, but CRLF2/P2RY8 intrachromosomal deletions\ndo not.\n\nTsai AG, Yoda A, Weinstock DM, Lieber MR.\n\nDOI: 10.1182/blood-2010-05-286492 \nPMCID: PMC5773278\nPMID: 20847213  [Indexed for MEDLINE]',
     '49. Blood. 2010 Jul 1;115(26):5393-7. doi: 10.1182/blood-2009-11-256131. Epub 2010\nApr 8.\n\nPresence of the P2RY8-CRLF2 rearrangement is associated with a poor prognosis in \nnon-high-risk precursor B-cell acute lymphoblastic leukemia in children treated\naccording to the ALL-BFM 2000 protocol.\n\nCario G(1), Zimmermann M, Romey R, Gesk S, Vater I, Harbott J, Schrauder A,\nMoericke A, Izraeli S, Akasaka T, Dyer MJ, Siebert R, Schrappe M, Stanulla M.\n\nAuthor information: \n(1)Department of Pediatrics, University Medical Center Schleswig-Holstein, Campus\nKiel, Arnold-Heller-Strasse 3, Kiel, Germany. g.cario@pediatrics.uni-kiel.de\n\nHigh-level expression of the cytokine receptor-like factor 2 gene, CRLF2, in\nprecursor B-cell acute lymphoblastic leukemia (pB-ALL) was shown to be caused by \na translocation involving the IGH@ locus or a deletion juxtaposing CRLF2 with the\nP2RY8 promoter. To assess its possible prognostic value, CRLF2 expression was\nanalyzed in 555 childhood pB-ALL patients treated according to the Acute\nLymphoblastic Leukemia Berlin-Frankfurt-Münster 2000 (ALL-BFM 2000) protocol.\nBesides CRLF2 rearrangements, high-level CRLF2 expression was seen in cases with \nsupernumerary copies of the CRLF2 locus. On the basis of the detection of CRLF2\nrearrangements, a CRLF2 high-expression group (n = 49) was defined. This group\nhad a 6-year relapse incidence of 31% plus or minus 8% compared with 11% plus or \nminus 1% in the CRLF2 low-expression group (P = .006). This difference was mainly\nattributable to an extremely high incidence of relapse (71% +/- 19%) in\nnon-high-risk patients with P2RY8-CRLF2 rearrangement. The assessment of CRLF2\naberrations may therefore serve as new stratification tool in\nBerlin-Frankfurt-Münster-based protocols by identifying additional high-risk\npatients who may benefit from an intensified and/or targeted treatment.\n\nDOI: 10.1182/blood-2009-11-256131 \nPMID: 20378752  [Indexed for MEDLINE]',
     '50. Blood. 2010 Jul 1;115(26):5312-21. doi: 10.1182/blood-2009-09-245944. Epub 2010\nFeb 4.\n\nRearrangement of CRLF2 is associated with mutation of JAK kinases, alteration of \nIKZF1, Hispanic/Latino ethnicity, and a poor outcome in pediatric B-progenitor\nacute lymphoblastic leukemia.\n\nHarvey RC(1), Mullighan CG, Chen IM, Wharton W, Mikhail FM, Carroll AJ, Kang H,\nLiu W, Dobbin KK, Smith MA, Carroll WL, Devidas M, Bowman WP, Camitta BM, Reaman \nGH, Hunger SP, Downing JR, Willman CL.\n\nAuthor information: \n(1)University of New Mexico Cancer Center and Departments of Pathology and\nInternal Medicine, University of New Mexico, Albuquerque, NM, USA.\n\nGene expression profiling of 207 uniformly treated children with high-risk\nB-progenitor acute lymphoblastic leukemia revealed 29 of 207 cases (14%) with\nmarkedly elevated expression of CRLF2 (cytokine receptor-like factor 2). Each of \nthe 29 cases harbored a genomic rearrangement of CRLF2: 18 of 29 (62%) had a\ntranslocation of the immunoglobulin heavy chain gene IGH@ on 14q32 to CRLF2 in\nthe pseudoautosomal region 1 of Xp22.3/Yp11.3, whereas 10 (34%) cases had a\n320-kb interstitial deletion centromeric of CRLF2, resulting in a P2RY8-CRLF2\nfusion. One case had both IGH@-CRLF2 and P2RY8-CRLF2, and another had a novel\nCRLF2 rearrangement. Only 2 of 29 cases were Down syndrome. CRLF2 rearrangements \nwere significantly associated with activating mutations of JAK1 or JAK2, deletion\nor mutation of IKZF1, and Hispanic/Latino ethnicity (Fisher exact test, P < .001 \nfor each). Within this cohort, patients with CRLF2 rearrangements had extremely\npoor treatment outcomes compared with those without CRLF2 rearrangements (35.3%\nvs 71.3% relapse-free survival at 4 years; P < .001). Together, these\nobservations suggest that activation of CRLF2 expression, mutation of JAK\nkinases, and alterations of IKZF1 cooperate to promote B-cell leukemogenesis and \nidentify these pathways as important therapeutic targets in this disease.\n\nDOI: 10.1182/blood-2009-09-245944 \nPMCID: PMC2902132\nPMID: 20139093  [Indexed for MEDLINE]\n\n']




```python
test_chunk = chunks[1]
```


```python
test_chunk
```




    '2. Immunol Rev. 2019 May;289(1):158-172. doi: 10.1111/imr.12743.\n\nG-protein coupled receptors and ligands that organize humoral immune responses.\n\nLu E(1), Cyster JG(1).\n\nAuthor information: \n(1)Howard Hughes Medical Institute and Department of Microbiology and Immunology,\nUniversity of California San Francisco, San Francisco, California.\n\nB-cell responses are dynamic processes that depend on multiple types of\ninteractions. Rare antigen-specific B cells must encounter antigen and\nspecialized systems are needed-unique to each lymphoid tissue type-to ensure this\nhappens efficiently. Lymphoid tissue barrier cells act to ensure that pathogens, \nwhile being permitted entry for B-cell recognition, are blocked from replication \nor dissemination. T follicular helper (Tfh) cells often need to be primed by\ndendritic cells before supporting B-cell responses. For most responses,\nantigen-specific helper T cells and B cells need to interact, first to initiate\nclonal expansion and the plasmablast response, and later to support the germinal \ncenter (GC) response. Newly formed plasma cells need to travel to supportive\nniches. GC B cells must become confined to the follicle center, organize into\ndark and light zones, and interact with Tfh cells. Memory B cells need to be\npositioned for rapid responses following reinfection. Each of these events\nrequires the actions of multiple G-protein coupled receptors (GPCRs) and their\nligands, including chemokines and lipid mediators. This review will focus on the \nguidance cue code underlying B-cell immunity, with an emphasis on findings from\nour laboratory and on newer advances in related areas. We will discuss our recent\nidentification of geranylgeranyl-glutathione as a ligand for P2RY8. Our goal is\nto provide the reader with a focused knowledge about the GPCRs guiding B-cell\nresponses and how they might be therapeutic targets, while also providing\nexamples of how multiple types of GPCRs can cooperate or act iteratively to\ncontrol cell behavior.\n\n© 2019 John Wiley & Sons A/S. Published by John Wiley & Sons Ltd.\n\nDOI: 10.1111/imr.12743 \nPMCID: PMC6464390 [Available on 2020-05-01]\nPMID: 30977196  [Indexed for MEDLINE]'




```python
splitchunk = test_chunk.split("\n\n")
```


```python
splitchunk
```




    ['2. Immunol Rev. 2019 May;289(1):158-172. doi: 10.1111/imr.12743.',
     'G-protein coupled receptors and ligands that organize humoral immune responses.',
     'Lu E(1), Cyster JG(1).',
     'Author information: \n(1)Howard Hughes Medical Institute and Department of Microbiology and Immunology,\nUniversity of California San Francisco, San Francisco, California.',
     'B-cell responses are dynamic processes that depend on multiple types of\ninteractions. Rare antigen-specific B cells must encounter antigen and\nspecialized systems are needed-unique to each lymphoid tissue type-to ensure this\nhappens efficiently. Lymphoid tissue barrier cells act to ensure that pathogens, \nwhile being permitted entry for B-cell recognition, are blocked from replication \nor dissemination. T follicular helper (Tfh) cells often need to be primed by\ndendritic cells before supporting B-cell responses. For most responses,\nantigen-specific helper T cells and B cells need to interact, first to initiate\nclonal expansion and the plasmablast response, and later to support the germinal \ncenter (GC) response. Newly formed plasma cells need to travel to supportive\nniches. GC B cells must become confined to the follicle center, organize into\ndark and light zones, and interact with Tfh cells. Memory B cells need to be\npositioned for rapid responses following reinfection. Each of these events\nrequires the actions of multiple G-protein coupled receptors (GPCRs) and their\nligands, including chemokines and lipid mediators. This review will focus on the \nguidance cue code underlying B-cell immunity, with an emphasis on findings from\nour laboratory and on newer advances in related areas. We will discuss our recent\nidentification of geranylgeranyl-glutathione as a ligand for P2RY8. Our goal is\nto provide the reader with a focused knowledge about the GPCRs guiding B-cell\nresponses and how they might be therapeutic targets, while also providing\nexamples of how multiple types of GPCRs can cooperate or act iteratively to\ncontrol cell behavior.',
     '© 2019 John Wiley & Sons A/S. Published by John Wiley & Sons Ltd.',
     'DOI: 10.1111/imr.12743 \nPMCID: PMC6464390 [Available on 2020-05-01]\nPMID: 30977196  [Indexed for MEDLINE]']


